No '-XX:...' jvm parameters are set. Adding safer GC settings '-XX:ParallelGCThreads=8' to the OZONE_OPTS
2025-06-17 21:40:47,263 [main] INFO recon.ReconServer: STARTUP_MSG: 
/************************************************************
STARTUP_MSG: Starting ReconServer
STARTUP_MSG:   host = recon/172.25.0.115
STARTUP_MSG:   args = []
STARTUP_MSG:   version = 2.1.0-SNAPSHOT
STARTUP_MSG:   classpath = /etc/hadoop:/opt/hadoop/share/ozone/lib/aopalliance-1.0.jar:/opt/hadoop/share/ozone/lib/jackson-annotations-2.16.2.jar:/opt/hadoop/share/ozone/lib/jackson-databind-2.16.2.jar:/opt/hadoop/share/ozone/lib/jackson-core-2.16.2.jar:/opt/hadoop/share/ozone/lib/guava-32.1.3-jre.jar:/opt/hadoop/share/ozone/lib/failureaccess-1.0.1.jar:/opt/hadoop/share/ozone/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/opt/hadoop/share/ozone/lib/checker-qual-3.37.0.jar:/opt/hadoop/share/ozone/lib/error_prone_annotations-2.29.2.jar:/opt/hadoop/share/ozone/lib/j2objc-annotations-2.8.jar:/opt/hadoop/share/ozone/lib/guice-6.0.0.jar:/opt/hadoop/share/ozone/lib/jakarta.inject-api-2.0.1.jar:/opt/hadoop/share/ozone/lib/guice-servlet-6.0.0.jar:/opt/hadoop/share/ozone/lib/protobuf-java-2.5.0.jar:/opt/hadoop/share/ozone/lib/bonecp-0.8.0.RELEASE.jar:/opt/hadoop/share/ozone/lib/commons-collections-3.2.2.jar:/opt/hadoop/share/ozone/lib/commons-io-2.18.0.jar:/opt/hadoop/share/ozone/lib/picocli-4.7.7.jar:/opt/hadoop/share/ozone/lib/jakarta.annotation-api-2.1.1.jar:/opt/hadoop/share/ozone/lib/jakarta.validation-api-2.0.2.jar:/opt/hadoop/share/ozone/lib/jakarta.ws.rs-api-2.1.6.jar:/opt/hadoop/share/ozone/lib/jakarta.xml.bind-api-2.3.3.jar:/opt/hadoop/share/ozone/lib/jakarta.activation-api-1.2.2.jar:/opt/hadoop/share/ozone/lib/javax.annotation-api-1.3.2.jar:/opt/hadoop/share/ozone/lib/javax.inject-1.jar:/opt/hadoop/share/ozone/lib/javax.servlet-api-3.1.0.jar:/opt/hadoop/share/ozone/lib/commons-compress-1.27.1.jar:/opt/hadoop/share/ozone/lib/commons-codec-1.17.1.jar:/opt/hadoop/share/ozone/lib/commons-lang3-3.17.0.jar:/opt/hadoop/share/ozone/lib/derby-10.14.2.0.jar:/opt/hadoop/share/ozone/lib/hadoop-auth-3.4.1.jar:/opt/hadoop/share/ozone/lib/httpclient-4.5.14.jar:/opt/hadoop/share/ozone/lib/nimbus-jose-jwt-9.40.jar:/opt/hadoop/share/ozone/lib/metrics-core-3.2.6.jar:/opt/hadoop/share/ozone/lib/kerb-core-1.0.1.jar:/opt/hadoop/share/ozone/lib/kerby-pkix-1.0.1.jar:/opt/hadoop/share/ozone/lib/kerby-asn1-1.0.1.jar:/opt/hadoop/share/ozone/lib/kerb-util-1.0.1.jar:/opt/hadoop/share/ozone/lib/kerby-config-1.0.1.jar:/opt/hadoop/share/ozone/lib/kerb-crypto-1.0.1.jar:/opt/hadoop/share/ozone/lib/hadoop-shaded-guava-1.3.0.jar:/opt/hadoop/share/ozone/lib/hadoop-common-3.4.1.jar:/opt/hadoop/share/ozone/lib/hadoop-shaded-protobuf_3_25-1.4.0.jar:/opt/hadoop/share/ozone/lib/hadoop-annotations-3.4.1.jar:/opt/hadoop/share/ozone/lib/commons-cli-1.8.0.jar:/opt/hadoop/share/ozone/lib/commons-net-3.11.1.jar:/opt/hadoop/share/ozone/lib/jetty-server-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-io-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-webapp-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-xml-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jsp-api-2.1.jar:/opt/hadoop/share/ozone/lib/jettison-1.5.4.jar:/opt/hadoop/share/ozone/lib/commons-beanutils-1.11.0.jar:/opt/hadoop/share/ozone/lib/commons-configuration2-2.11.0.jar:/opt/hadoop/share/ozone/lib/commons-text-1.12.0.jar:/opt/hadoop/share/ozone/lib/re2j-1.7.jar:/opt/hadoop/share/ozone/lib/gson-2.10.1.jar:/opt/hadoop/share/ozone/lib/jsch-0.1.55.jar:/opt/hadoop/share/ozone/lib/netty-handler-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-resolver-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-transport-native-unix-common-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-transport-native-epoll-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-transport-classes-epoll-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/bcprov-jdk18on-1.81.jar:/opt/hadoop/share/ozone/lib/stax2-api-4.2.1.jar:/opt/hadoop/share/ozone/lib/woodstox-core-5.4.0.jar:/opt/hadoop/share/ozone/lib/dnsjava-3.6.1.jar:/opt/hadoop/share/ozone/lib/hadoop-hdfs-client-3.4.1.jar:/opt/hadoop/share/ozone/lib/httpcore-4.4.16.jar:/opt/hadoop/share/ozone/lib/hdds-common-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/jackson-datatype-jsr310-2.16.2.jar:/opt/hadoop/share/ozone/lib/jcip-annotations-1.0-1.jar:/opt/hadoop/share/ozone/lib/commons-validator-1.6.jar:/opt/hadoop/share/ozone/lib/commons-digester-1.8.1.jar:/opt/hadoop/share/ozone/lib/jaeger-core-1.8.1.jar:/opt/hadoop/share/ozone/lib/opentracing-api-0.33.0.jar:/opt/hadoop/share/ozone/lib/opentracing-util-0.33.0.jar:/opt/hadoop/share/ozone/lib/opentracing-noop-0.33.0.jar:/opt/hadoop/share/ozone/lib/hdds-interface-admin-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ratis-client-3.1.2.jar:/opt/hadoop/share/ozone/lib/ratis-grpc-3.1.2.jar:/opt/hadoop/share/ozone/lib/ratis-netty-3.1.2.jar:/opt/hadoop/share/ozone/lib/ratis-server-api-3.1.2.jar:/opt/hadoop/share/ozone/lib/ratis-thirdparty-misc-1.0.6.jar:/opt/hadoop/share/ozone/lib/bcpkix-jdk18on-1.81.jar:/opt/hadoop/share/ozone/lib/bcutil-jdk18on-1.81.jar:/opt/hadoop/share/ozone/lib/snakeyaml-2.0.jar:/opt/hadoop/share/ozone/lib/jaeger-client-1.8.1.jar:/opt/hadoop/share/ozone/lib/jaeger-thrift-1.8.1.jar:/opt/hadoop/share/ozone/lib/libthrift-0.15.0.jar:/opt/hadoop/share/ozone/lib/okhttp-4.12.0.jar:/opt/hadoop/share/ozone/lib/okio-3.6.0.jar:/opt/hadoop/share/ozone/lib/okio-jvm-3.6.0.jar:/opt/hadoop/share/ozone/lib/kotlin-stdlib-common-1.9.25.jar:/opt/hadoop/share/ozone/lib/kotlin-stdlib-jdk8-1.9.25.jar:/opt/hadoop/share/ozone/lib/kotlin-stdlib-1.9.25.jar:/opt/hadoop/share/ozone/lib/annotations-13.0.jar:/opt/hadoop/share/ozone/lib/kotlin-stdlib-jdk7-1.9.25.jar:/opt/hadoop/share/ozone/lib/jaeger-tracerresolver-1.8.1.jar:/opt/hadoop/share/ozone/lib/opentracing-tracerresolver-0.1.8.jar:/opt/hadoop/share/ozone/lib/ratis-metrics-dropwizard3-3.1.2.jar:/opt/hadoop/share/ozone/lib/jcl-over-slf4j-2.0.17.jar:/opt/hadoop/share/ozone/lib/slf4j-reload4j-2.0.17.jar:/opt/hadoop/share/ozone/lib/hdds-config-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/hdds-container-service-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/netty-buffer-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-codec-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-common-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-transport-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/hadoop-hdfs-3.4.1.jar:/opt/hadoop/share/ozone/lib/commons-daemon-1.4.0.jar:/opt/hadoop/share/ozone/lib/log4j-api-2.24.2.jar:/opt/hadoop/share/ozone/lib/hdds-client-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/hdds-erasurecode-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ratis-server-3.1.2.jar:/opt/hadoop/share/ozone/lib/zstd-jni-1.5.7-3.jar:/opt/hadoop/share/ozone/lib/apache-log4j-extras-1.2.17.jar:/opt/hadoop/share/ozone/lib/jaxb-runtime-2.3.9.jar:/opt/hadoop/share/ozone/lib/txw2-2.3.9.jar:/opt/hadoop/share/ozone/lib/istack-commons-runtime-3.0.12.jar:/opt/hadoop/share/ozone/lib/jakarta.activation-1.2.2.jar:/opt/hadoop/share/ozone/lib/hdds-interface-client-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/hdds-interface-server-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/hdds-managed-rocksdb-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/hdds-server-framework-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/reload4j-1.2.26.jar:/opt/hadoop/share/ozone/lib/jnr-constants-0.10.4.jar:/opt/hadoop/share/ozone/lib/jnr-posix-3.1.20.jar:/opt/hadoop/share/ozone/lib/jnr-ffi-2.2.17.jar:/opt/hadoop/share/ozone/lib/jffi-1.3.13.jar:/opt/hadoop/share/ozone/lib/jffi-1.3.13-native.jar:/opt/hadoop/share/ozone/lib/asm-9.7.1.jar:/opt/hadoop/share/ozone/lib/asm-commons-9.7.1.jar:/opt/hadoop/share/ozone/lib/asm-analysis-9.7.1.jar:/opt/hadoop/share/ozone/lib/asm-tree-9.7.1.jar:/opt/hadoop/share/ozone/lib/asm-util-9.7.1.jar:/opt/hadoop/share/ozone/lib/jnr-a64asm-1.0.0.jar:/opt/hadoop/share/ozone/lib/jnr-x86asm-1.0.2.jar:/opt/hadoop/share/ozone/lib/commons-fileupload-1.5.jar:/opt/hadoop/share/ozone/lib/grpc-api-1.58.0.jar:/opt/hadoop/share/ozone/lib/simpleclient-0.16.0.jar:/opt/hadoop/share/ozone/lib/simpleclient_common-0.16.0.jar:/opt/hadoop/share/ozone/lib/simpleclient_dropwizard-0.16.0.jar:/opt/hadoop/share/ozone/lib/rocksdb-checkpoint-differ-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ratis-metrics-api-3.1.2.jar:/opt/hadoop/share/ozone/lib/jetty-http-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/disruptor-3.4.4.jar:/opt/hadoop/share/ozone/lib/log4j-core-2.24.2.jar:/opt/hadoop/share/ozone/lib/snappy-java-1.1.10.7.jar:/opt/hadoop/share/ozone/lib/hdds-server-scm-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ozone-cli-admin-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ozone-common-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/grpc-netty-1.58.0.jar:/opt/hadoop/share/ozone/lib/grpc-core-1.58.0.jar:/opt/hadoop/share/ozone/lib/annotations-4.1.1.4.jar:/opt/hadoop/share/ozone/lib/animal-sniffer-annotations-1.23.jar:/opt/hadoop/share/ozone/lib/grpc-context-1.58.0.jar:/opt/hadoop/share/ozone/lib/grpc-util-1.58.0.jar:/opt/hadoop/share/ozone/lib/perfmark-api-0.26.0.jar:/opt/hadoop/share/ozone/lib/netty-codec-http2-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-codec-http-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-handler-proxy-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-codec-socks-4.1.109.Final.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-classes-2.0.65.Final.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final-linux-x86_64.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final-linux-aarch_64.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final-osx-x86_64.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final-osx-aarch_64.jar:/opt/hadoop/share/ozone/lib/netty-tcnative-boringssl-static-2.0.65.Final-windows-x86_64.jar:/opt/hadoop/share/ozone/lib/ozone-interface-client-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/grpc-protobuf-1.58.0.jar:/opt/hadoop/share/ozone/lib/proto-google-common-protos-2.22.0.jar:/opt/hadoop/share/ozone/lib/grpc-protobuf-lite-1.58.0.jar:/opt/hadoop/share/ozone/lib/grpc-stub-1.58.0.jar:/opt/hadoop/share/ozone/lib/ozone-interface-storage-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ozone-manager-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/kerby-util-1.0.1.jar:/opt/hadoop/share/ozone/lib/hdds-rocks-native-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/ranger-intg-2.6.0.jar:/opt/hadoop/share/ozone/lib/ranger-plugins-common-2.6.0.jar:/opt/hadoop/share/ozone/lib/jackson-jaxrs-base-2.16.2.jar:/opt/hadoop/share/ozone/lib/jackson-jaxrs-json-provider-2.16.2.jar:/opt/hadoop/share/ozone/lib/gethostname4j-1.0.0.jar:/opt/hadoop/share/ozone/lib/commons-lang-2.6.jar:/opt/hadoop/share/ozone/lib/jna-5.2.0.jar:/opt/hadoop/share/ozone/lib/jna-platform-5.2.0.jar:/opt/hadoop/share/ozone/lib/ranger-plugin-classloader-2.6.0.jar:/opt/hadoop/share/ozone/lib/ranger-plugins-audit-2.6.0.jar:/opt/hadoop/share/ozone/lib/aws-java-sdk-logs-1.12.765.jar:/opt/hadoop/share/ozone/lib/aws-java-sdk-core-1.12.661.jar:/opt/hadoop/share/ozone/lib/jackson-dataformat-cbor-2.16.2.jar:/opt/hadoop/share/ozone/lib/jmespath-java-1.12.765.jar:/opt/hadoop/share/ozone/lib/hppc-0.8.0.jar:/opt/hadoop/share/ozone/lib/aircompressor-0.27.jar:/opt/hadoop/share/ozone/lib/joda-time-2.12.7.jar:/opt/hadoop/share/ozone/lib/httpasyncclient-4.1.4.jar:/opt/hadoop/share/ozone/lib/httpcore-nio-4.4.16.jar:/opt/hadoop/share/ozone/lib/httpmime-4.5.13.jar:/opt/hadoop/share/ozone/lib/orc-core-1.5.8.jar:/opt/hadoop/share/ozone/lib/orc-shims-1.5.8.jar:/opt/hadoop/share/ozone/lib/jetty-client-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/ranger-plugins-cred-2.6.0.jar:/opt/hadoop/share/ozone/lib/aspectjrt-1.9.7.jar:/opt/hadoop/share/ozone/lib/ozone-reconcodegen-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/lib/jooq-codegen-3.11.10.jar:/opt/hadoop/share/ozone/lib/jooq-meta-3.11.10.jar:/opt/hadoop/share/ozone/lib/ratis-common-3.1.2.jar:/opt/hadoop/share/ozone/lib/ratis-proto-3.1.2.jar:/opt/hadoop/share/ozone/lib/jetty-servlet-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-security-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-util-ajax-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/jetty-util-9.4.57.v20241219.jar:/opt/hadoop/share/ozone/lib/guice-bridge-2.6.1.jar:/opt/hadoop/share/ozone/lib/hk2-api-2.6.1.jar:/opt/hadoop/share/ozone/lib/hk2-utils-2.6.1.jar:/opt/hadoop/share/ozone/lib/aopalliance-repackaged-2.6.1.jar:/opt/hadoop/share/ozone/lib/jersey-container-servlet-core-2.47.jar:/opt/hadoop/share/ozone/lib/jersey-common-2.47.jar:/opt/hadoop/share/ozone/lib/osgi-resource-locator-1.0.3.jar:/opt/hadoop/share/ozone/lib/jersey-server-2.47.jar:/opt/hadoop/share/ozone/lib/jersey-client-2.47.jar:/opt/hadoop/share/ozone/lib/jooq-3.11.10.jar:/opt/hadoop/share/ozone/lib/reflections-0.10.2.jar:/opt/hadoop/share/ozone/lib/javassist-3.30.2-GA.jar:/opt/hadoop/share/ozone/lib/rocksdbjni-7.7.3.jar:/opt/hadoop/share/ozone/lib/slf4j-api-2.0.17.jar:/opt/hadoop/share/ozone/lib/spring-core-5.3.39.jar:/opt/hadoop/share/ozone/lib/spring-jdbc-5.3.39.jar:/opt/hadoop/share/ozone/lib/spring-beans-5.3.39.jar:/opt/hadoop/share/ozone/lib/spring-tx-5.3.39.jar:/opt/hadoop/share/ozone/lib/sqlite-jdbc-3.50.1.0.jar:/opt/hadoop/share/ozone/lib/guice-assistedinject-6.0.0.jar:/opt/hadoop/share/ozone/lib/jakarta.inject-2.6.1.jar:/opt/hadoop/share/ozone/lib/jersey-hk2-2.47.jar:/opt/hadoop/share/ozone/lib/hk2-locator-2.6.1.jar:/opt/hadoop/share/ozone/lib/jersey-media-jaxb-2.47.jar:/opt/hadoop/share/ozone/lib/jersey-media-json-jackson-2.47.jar:/opt/hadoop/share/ozone/lib/jersey-entity-filtering-2.47.jar:/opt/hadoop/share/ozone/lib/jackson-module-jaxb-annotations-2.16.2.jar:/opt/hadoop/share/ozone/lib/ozone-recon-2.1.0-SNAPSHOT.jar:/opt/hadoop/share/ozone/web
STARTUP_MSG:   build = https://github.com/apache/ozone/6df3077fe19d2280a6bba34d9c10c6f00eb92ebc
STARTUP_MSG:   java = 21.0.2
STARTUP_MSG:   conf = {hadoop.hdds.db.rocksdb.WAL_size_limit_MB=0MB, hadoop.hdds.db.rocksdb.WAL_ttl_seconds=1200, hadoop.hdds.db.rocksdb.keep.log.file.num=10, hadoop.hdds.db.rocksdb.logging.enabled=false, hadoop.hdds.db.rocksdb.logging.level=INFO, hadoop.hdds.db.rocksdb.max.log.file.size=100MB, hadoop.hdds.db.rocksdb.writeoption.sync=false, hdds.block.token.enabled=true, hdds.block.token.expiry.time=1d, hdds.command.status.report.interval=30s, hdds.container.action.max.limit=20, hdds.container.balancer.balancing.iteration.interval=70m, hdds.container.balancer.datanodes.involved.max.percentage.per.iteration=20, hdds.container.balancer.iterations=10, hdds.container.balancer.move.networkTopology.enable=false, hdds.container.balancer.move.replication.timeout=50m, hdds.container.balancer.move.timeout=65m, hdds.container.balancer.size.entering.target.max=26GB, hdds.container.balancer.size.leaving.source.max=26GB, hdds.container.balancer.size.moved.max.per.iteration=500GB, hdds.container.balancer.trigger.du.before.move.enable=false, hdds.container.balancer.utilization.threshold=10, hdds.container.checksum.verification.enabled=true, hdds.container.chunk.write.sync=false, hdds.container.close.threshold=0.9f, hdds.container.ipc.port=9859, hdds.container.ipc.random.port=false, hdds.container.ratis.admin.port=9857, hdds.container.ratis.datastream.enabled=true, hdds.container.ratis.datastream.port=9855, hdds.container.ratis.datastream.random.port=false, hdds.container.ratis.enabled=false, hdds.container.ratis.ipc.port=9858, hdds.container.ratis.ipc.random.port=false, hdds.container.ratis.leader.pending.bytes.limit=1GB, hdds.container.ratis.log.appender.queue.byte-limit=32MB, hdds.container.ratis.log.appender.queue.num-elements=1024, hdds.container.ratis.log.purge.gap=1000000, hdds.container.ratis.log.queue.byte-limit=4GB, hdds.container.ratis.log.queue.num-elements=1024, hdds.container.ratis.num.container.op.executors=10, hdds.container.ratis.num.write.chunk.threads.per.volume=10, hdds.container.ratis.rpc.type=GRPC, hdds.container.ratis.segment.preallocated.size=4MB, hdds.container.ratis.segment.size=64MB, hdds.container.ratis.server.port=9856, hdds.container.ratis.statemachine.max.pending.apply-transactions=100000, hdds.container.ratis.statemachine.write.wait.interval=10m, hdds.container.ratis.statemachinedata.sync.timeout=10s, hdds.container.replication.compression=NO_COMPRESSION, hdds.container.report.interval=60s, hdds.container.scrub.data.scan.interval=7d, hdds.container.scrub.dev.data.scan.enabled=true, hdds.container.scrub.dev.metadata.scan.enabled=true, hdds.container.scrub.enabled=true, hdds.container.scrub.metadata.scan.interval=3h, hdds.container.scrub.min.gap=15m, hdds.container.scrub.on.demand.volume.bytes.per.second=5242880, hdds.container.scrub.volume.bytes.per.second=5242880, hdds.container.token.enabled=true, hdds.datanode.block.delete.command.worker.interval=2s, hdds.datanode.block.delete.max.lock.wait.timeout=100ms, hdds.datanode.block.delete.queue.limit=5, hdds.datanode.block.delete.threads.max=5, hdds.datanode.block.deleting.limit.per.interval=5000, hdds.datanode.block.deleting.max.lock.holding.time=1s, hdds.datanode.block.deleting.service.interval=60s, hdds.datanode.check.empty.container.dir.on.delete=false, hdds.datanode.chunk.data.validation.check=false, hdds.datanode.client.bind.host=0.0.0.0, hdds.datanode.client.port=19864, hdds.datanode.command.queue.limit=5000, hdds.datanode.container.close.threads.max=3, hdds.datanode.container.delete.threads.max=2, hdds.datanode.container.schema.v3.enabled=true, hdds.datanode.container.schema.v3.key.separator=|, hdds.datanode.delete.container.timeout=60s, hdds.datanode.df.refresh.period=5m, hdds.datanode.dir=/data/hdds, hdds.datanode.disk.check.io.failures.tolerated=1, hdds.datanode.disk.check.io.file.size=100B, hdds.datanode.disk.check.io.test.count=3, hdds.datanode.disk.check.min.gap=10m, hdds.datanode.disk.check.timeout=10m, hdds.datanode.dns.interface=default, hdds.datanode.dns.nameserver=default, hdds.datanode.du.refresh.period=1h, hdds.datanode.failed.data.volumes.tolerated=-1, hdds.datanode.failed.db.volumes.tolerated=-1, hdds.datanode.failed.metadata.volumes.tolerated=-1, hdds.datanode.handler.count=10, hdds.datanode.http-address=0.0.0.0:9882, hdds.datanode.http-bind-host=0.0.0.0, hdds.datanode.http.auth.kerberos.keytab=/etc/security/keytabs/HTTP.keytab, hdds.datanode.http.auth.kerberos.principal=HTTP/db@EXAMPLE.COM, hdds.datanode.http.auth.type=kerberos, hdds.datanode.http.enabled=true, hdds.datanode.https-address=0.0.0.0:9883, hdds.datanode.https-bind-host=0.0.0.0, hdds.datanode.kerberos.keytab.file=/etc/security/keytabs/dn.keytab, hdds.datanode.kerberos.principal=dn/dn@EXAMPLE.COM, hdds.datanode.metadata.rocksdb.cache.size=1GB, hdds.datanode.periodic.disk.check.interval.minutes=60, hdds.datanode.ratis.server.request.timeout=2m, hdds.datanode.read.chunk.threads.per.volume=10, hdds.datanode.read.threadpool=10, hdds.datanode.recovering.container.scrubbing.service.interval=1m, hdds.datanode.replication.outofservice.limit.factor=2.0, hdds.datanode.replication.port=9886, hdds.datanode.replication.queue.limit=4096, hdds.datanode.replication.streams.limit=10, hdds.datanode.rocksdb.auto-compaction-small-sst-file=true, hdds.datanode.rocksdb.auto-compaction-small-sst-file-num-threshold=512, hdds.datanode.rocksdb.auto-compaction-small-sst-file-size-threshold=1MB, hdds.datanode.rocksdb.auto-compaction-small-sst-file.interval.minutes=120, hdds.datanode.rocksdb.auto-compaction-small-sst-file.threads=1, hdds.datanode.rocksdb.delete-obsolete-files-period=1h, hdds.datanode.rocksdb.log.level=INFO, hdds.datanode.rocksdb.log.max-file-num=64, hdds.datanode.rocksdb.log.max-file-size=32MB, hdds.datanode.rocksdb.max-open-files=1024, hdds.datanode.slow.op.warning.threshold=500ms, hdds.datanode.storage.utilization.critical.threshold=0.95, hdds.datanode.storage.utilization.warning.threshold=0.75, hdds.datanode.use.datanode.hostname=false, hdds.datanode.volume.choosing.policy=org.apache.hadoop.ozone.container.common.volume.CapacityVolumeChoosingPolicy, hdds.datanode.volume.min.free.space=100MB, hdds.datanode.volume.min.free.space.percent=-1, hdds.datanode.wait.on.all.followers=false, hdds.db.profile=DISK, hdds.grpc.tls.enabled=true, hdds.grpc.tls.provider=OPENSSL, hdds.heartbeat.initial-interval=2s, hdds.heartbeat.interval=30s, hdds.heartbeat.recon.initial-interval=60s, hdds.heartbeat.recon.interval=60s, hdds.key.algo=RSA, hdds.key.dir.name=keys, hdds.key.len=2048, hdds.node.report.interval=60000ms, hdds.pipeline.action.max.limit=20, hdds.pipeline.report.interval=60000ms, hdds.priv.key.file.name=private.pem, hdds.profiler.endpoint.enabled=false, hdds.prometheus.endpoint.enabled=true, hdds.public.key.file.name=public.pem, hdds.ratis.client.exponential.backoff.base.sleep=4s, hdds.ratis.client.exponential.backoff.max.retries=2147483647, hdds.ratis.client.exponential.backoff.max.sleep=40s, hdds.ratis.client.multilinear.random.retry.policy=5s, 5, 10s, 5, 15s, 5, 20s, 5, 25s, 5, 60s, 10, hdds.ratis.client.request.watch.timeout=3m, hdds.ratis.client.request.watch.type=ALL_COMMITTED, hdds.ratis.client.request.write.timeout=5m, hdds.ratis.client.retry.policy=org.apache.hadoop.hdds.ratis.retrypolicy.RequestTypeDependentRetryPolicyCreator, hdds.ratis.client.retrylimited.max.retries=180, hdds.ratis.client.retrylimited.retry.interval=1s, hdds.ratis.leader.election.minimum.timeout.duration=5s, hdds.ratis.raft.client.async.outstanding-requests.max=32, hdds.ratis.raft.client.rpc.request.timeout=60s, hdds.ratis.raft.client.rpc.watch.request.timeout=180s, hdds.ratis.raft.grpc.flow.control.window=5MB, hdds.ratis.raft.server.datastream.client.pool.size=10, hdds.ratis.raft.server.datastream.request.threads=20, hdds.ratis.raft.server.delete.ratis.log.directory=true, hdds.ratis.raft.server.leaderelection.pre-vote=true, hdds.ratis.raft.server.log.appender.wait-time.min=0us, hdds.ratis.raft.server.notification.no-leader.timeout=300s, hdds.ratis.raft.server.rpc.request.timeout=60s, hdds.ratis.raft.server.rpc.slowness.timeout=300s, hdds.ratis.raft.server.watch.timeout=30s, hdds.ratis.raft.server.write.element-limit=1024, hdds.ratis.server.num.snapshots.retained=5, hdds.ratis.server.retry-cache.timeout.duration=600000ms, hdds.ratis.snapshot.threshold=100000, hdds.scm.block.deleting.service.interval=60s, hdds.scm.block.deletion.per-interval.max=100000, hdds.scm.block.deletion.txn.dn.commit.map.limit=5000000, hdds.scm.ec.pipeline.choose.policy.impl=org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RandomPipelineChoosePolicy, hdds.scm.http.auth.kerberos.keytab=/etc/security/keytabs/HTTP.keytab, hdds.scm.http.auth.kerberos.principal=HTTP/scm@EXAMPLE.COM, hdds.scm.http.auth.type=kerberos, hdds.scm.kerberos.keytab.file=/etc/security/keytabs/scm.keytab, hdds.scm.kerberos.principal=scm/scm@EXAMPLE.COM, hdds.scm.pipeline.choose.policy.impl=org.apache.hadoop.hdds.scm.pipeline.choose.algorithms.RandomPipelineChoosePolicy, hdds.scm.replication.datanode.delete.container.limit=40, hdds.scm.replication.datanode.reconstruction.weight=3, hdds.scm.replication.datanode.replication.limit=20, hdds.scm.replication.event.timeout=12m, hdds.scm.replication.event.timeout.datanode.offset=6m, hdds.scm.replication.inflight.limit.factor=0.75, hdds.scm.replication.maintenance.remaining.redundancy=1, hdds.scm.replication.maintenance.replica.minimum=2, hdds.scm.replication.over.replicated.interval=30s, hdds.scm.replication.push=true, hdds.scm.replication.thread.interval=300s, hdds.scm.replication.under.replicated.interval=30s, hdds.scm.safemode.atleast.one.node.reported.pipeline.pct=0.90, hdds.scm.safemode.enabled=true, hdds.scm.safemode.healthy.pipeline.pct=0.10, hdds.scm.safemode.min.datanode=3, hdds.scm.safemode.pipeline.creation=true, hdds.scm.safemode.threshold.pct=0.99, hdds.scm.unknown-container.action=WARN, hdds.scm.wait.time.after.safemode.exit=5m, hdds.scmclient.failover.max.retry=15, hdds.scmclient.failover.retry.interval=2s, hdds.scmclient.max.retry.timeout=30s, hdds.scmclient.rpc.timeout=15m, hdds.secret.key.algorithm=HmacSHA256, hdds.secret.key.expiry.duration=1h, hdds.secret.key.file.name=secret_keys.json, hdds.secret.key.rotate.check.duration=1m, hdds.secret.key.rotate.duration=5m, hdds.security.client.datanode.container.protocol.acl=*, hdds.security.client.scm.block.protocol.acl=*, hdds.security.client.scm.certificate.protocol.acl=*, hdds.security.client.scm.container.protocol.acl=*, hdds.security.client.scm.secretkey.datanode.protocol.acl=*, hdds.security.client.scm.secretkey.om.protocol.acl=*, hdds.security.client.scm.secretkey.scm.protocol.acl=*, hdds.security.provider=BC, hdds.tracing.enabled=false, hdds.x509.ca.rotation.ack.timeout=PT15M, hdds.x509.ca.rotation.check.interval=P1D, hdds.x509.ca.rotation.enabled=false, hdds.x509.ca.rotation.time-of-day=02:00:00, hdds.x509.default.duration=P365D, hdds.x509.dir.name=certs, hdds.x509.expired.certificate.check.interval=P1D, hdds.x509.file.name=certificate.crt, hdds.x509.max.duration=P1865D, hdds.x509.renew.grace.duration=P28D, hdds.x509.rootca.certificate.polling.interval=PT2h, hdds.x509.signature.algorithm=SHA256withRSA, hdds.xframe.enabled=true, hdds.xframe.value=SAMEORIGIN, ozone.UnsafeByteOperations.enabled=true, ozone.acl.authorizer.class=org.apache.hadoop.ozone.security.acl.OzoneNativeAuthorizer, ozone.acl.enabled=true, ozone.administrators=testuser,recon,om, ozone.block.deleting.service.interval=1m, ozone.block.deleting.service.timeout=300000ms, ozone.block.deleting.service.workers=10, ozone.chunk.read.buffer.default.size=1MB, ozone.chunk.read.mapped.buffer.max.count=0, ozone.chunk.read.mapped.buffer.threshold=32KB, ozone.client.bucket.replication.config.refresh.time.ms=30000, ozone.client.bytes.per.checksum=16KB, ozone.client.checksum.combine.mode=COMPOSITE_CRC, ozone.client.checksum.type=CRC32, ozone.client.connection.timeout=5000ms, ozone.client.datastream.buffer.flush.size=16MB, ozone.client.datastream.min.packet.size=1MB, ozone.client.datastream.pipeline.mode=true, ozone.client.datastream.window.size=64MB, ozone.client.ec.grpc.retries.enabled=true, ozone.client.ec.grpc.retries.max=3, ozone.client.ec.grpc.write.timeout=30s, ozone.client.ec.reconstruct.stripe.read.pool.limit=30, ozone.client.ec.reconstruct.stripe.write.pool.limit=30, ozone.client.ec.stripe.queue.size=2, ozone.client.exclude.nodes.expiry.time=600000, ozone.client.failover.max.attempts=500, ozone.client.fs.default.bucket.layout=FILE_SYSTEM_OPTIMIZED, ozone.client.hbase.enhancements.allowed=false, ozone.client.incremental.chunk.list=false, ozone.client.key.latest.version.location=true, ozone.client.key.provider.cache.expiry=10d, ozone.client.key.write.concurrency=1, ozone.client.list.cache=1000, ozone.client.max.ec.stripe.write.retries=10, ozone.client.max.retries=5, ozone.client.read.max.retries=3, ozone.client.read.retry.interval=1, ozone.client.read.timeout=30s, ozone.client.retry.interval=0, ozone.client.server-defaults.validity.period.ms=3600000, ozone.client.socket.timeout=5000ms, ozone.client.stream.buffer.flush.delay=true, ozone.client.stream.buffer.flush.size=16MB, ozone.client.stream.buffer.increment=0B, ozone.client.stream.buffer.max.size=32MB, ozone.client.stream.buffer.size=4MB, ozone.client.stream.putblock.piggybacking=false, ozone.client.verify.checksum=true, ozone.client.wait.between.retries.millis=2000, ozone.compaction.service.enabled=false, ozone.container.cache.lock.stripes=1024, ozone.container.cache.size=1024, ozone.directory.deleting.service.interval=1m, ozone.filesystem.snapshot.enabled=true, ozone.freon.http-address=0.0.0.0:9884, ozone.freon.http-bind-host=0.0.0.0, ozone.freon.http.auth.kerberos.keytab=/etc/security/keytabs/HTTP.keytab, ozone.freon.http.auth.kerberos.principal=HTTP/_HOST@REALM, ozone.freon.http.auth.type=simple, ozone.freon.http.enabled=true, ozone.freon.https-address=0.0.0.0:9885, ozone.freon.https-bind-host=0.0.0.0, ozone.fs.datastream.auto.threshold=4MB, ozone.fs.datastream.enabled=false, ozone.fs.hsync.enabled=false, ozone.fs.iterate.batch-size=100, ozone.fs.listing.page.size=1024, ozone.fs.listing.page.size.max=5000, ozone.handler.type=distributed, ozone.hbase.enhancements.allowed=false, ozone.http.basedir=/tmp/ozone_http, ozone.http.filter.initializers=org.apache.hadoop.security.AuthenticationFilterInitializer, ozone.http.policy=HTTP_ONLY, ozone.httpfs.http.auth.kerberos.keytab=/etc/security/keytabs/httpfs.keytab, ozone.httpfs.http.auth.kerberos.principal=HTTP/httpfs@EXAMPLE.COM, ozone.httpfs.http.auth.type=kerberos, ozone.httpfs.kerberos.keytab.file=/etc/security/keytabs/httpfs.keytab, ozone.httpfs.kerberos.principal=httpfs/httpfs@EXAMPLE.COM, ozone.https.client.keystore.resource=ssl-client.xml, ozone.https.client.need-auth=false, ozone.https.server.keystore.resource=ssl-server.xml, ozone.key.deleting.limit.per.task=20000, ozone.key.preallocation.max.blocks=64, ozone.manager.delegation.remover.scan.interval=3600000, ozone.manager.delegation.token.max-lifetime=7d, ozone.manager.delegation.token.renew-interval=1d, ozone.metadata.dirs=/data/metadata, ozone.metadata.dirs.permissions=750, ozone.metastore.rocksdb.cf.write.buffer.size=128MB, ozone.metastore.rocksdb.statistics=OFF, ozone.network.flexible.fqdn.resolution.enabled=false, ozone.network.jvm.address.cache.enabled=true, ozone.network.topology.aware.read=true, ozone.om.address=0.0.0.0:9862, ozone.om.address.omservice.om1=om1, ozone.om.address.omservice.om2=om2, ozone.om.address.omservice.om3=om3, ozone.om.admin.protocol.max.retries=20, ozone.om.admin.protocol.wait.between.retries=1000, ozone.om.client.rpc.timeout=15m, ozone.om.client.trash.core.pool.size=5, ozone.om.compaction.service.columnfamilies=keyTable,fileTable,directoryTable,deletedTable,deletedDirectoryTable,multipartInfoTable, ozone.om.compaction.service.run.interval=6h, ozone.om.compaction.service.timeout=10m, ozone.om.container.location.cache.size=100000, ozone.om.container.location.cache.ttl=360m, ozone.om.db.dirs.permissions=750, ozone.om.db.max.open.files=-1, ozone.om.delta.update.data.size.max.limit=1024MB, ozone.om.edekcacheloader.initial.delay.ms=3000, ozone.om.edekcacheloader.interval.ms=1000, ozone.om.edekcacheloader.max-retries=10, ozone.om.enable.filesystem.paths=false, ozone.om.enable.ofs.shared.tmp.dir=false, ozone.om.fs.snapshot.max.limit=10000, ozone.om.group.rights=READ, LIST, ozone.om.grpc.bossgroup.size=8, ozone.om.grpc.maximum.response.length=134217728, ozone.om.grpc.port=8981, ozone.om.grpc.read.thread.num=32, ozone.om.grpc.workergroup.size=32, ozone.om.ha.raft.server.log.appender.wait-time.min=0ms, ozone.om.ha.raft.server.retrycache.expirytime=300s, ozone.om.handler.count.key=100, ozone.om.http-address=0.0.0.0:9874, ozone.om.http-address.omservice.om1=om1, ozone.om.http-address.omservice.om2=om2, ozone.om.http-address.omservice.om3=om3, ozone.om.http-bind-host=0.0.0.0, ozone.om.http.auth.kerberos.keytab=/etc/security/keytabs/HTTP.keytab, ozone.om.http.auth.kerberos.principal=HTTP/om@EXAMPLE.COM, ozone.om.http.auth.type=kerberos, ozone.om.http.enabled=true, ozone.om.https-address=0.0.0.0:9875, ozone.om.https-bind-host=0.0.0.0, ozone.om.internal.service.id=omservice, ozone.om.kerberos.keytab.file=/etc/security/keytabs/om.keytab, ozone.om.kerberos.principal=om/om@EXAMPLE.COM, ozone.om.kerberos.principal.pattern=*, ozone.om.key.path.lock.enabled=false, ozone.om.keyname.character.check.enabled=false, ozone.om.leader.election.minimum.timeout.duration=5s, ozone.om.lease.hard.limit=7d, ozone.om.lease.soft.limit=60s, ozone.om.lock.fair=false, ozone.om.max.buckets=100000, ozone.om.multitenancy.enabled=false, ozone.om.multitenancy.ranger.sync.interval=10m, ozone.om.multitenancy.ranger.sync.timeout=10s, ozone.om.namespace.s3.strict=true, ozone.om.network.topology.refresh.duration=1h, ozone.om.nodes.omservice=om1,om2,om3, ozone.om.open.key.cleanup.limit.per.task=1000, ozone.om.open.key.cleanup.service.interval=24h, ozone.om.open.key.cleanup.service.timeout=300s, ozone.om.open.key.expire.threshold=7d, ozone.om.open.mpu.cleanup.service.interval=24h, ozone.om.open.mpu.cleanup.service.timeout=300s, ozone.om.open.mpu.expire.threshold=30d, ozone.om.open.mpu.parts.cleanup.limit.per.task=1000, ozone.om.ratis.log.appender.queue.byte-limit=32MB, ozone.om.ratis.log.appender.queue.num-elements=1024, ozone.om.ratis.log.purge.gap=1000000, ozone.om.ratis.log.purge.preservation.log.num=0, ozone.om.ratis.log.purge.upto.snapshot.index=true, ozone.om.ratis.minimum.timeout=5s, ozone.om.ratis.port=9872, ozone.om.ratis.rpc.type=GRPC, ozone.om.ratis.segment.preallocated.size=4MB, ozone.om.ratis.segment.size=4MB, ozone.om.ratis.server.close.threshold=60s, ozone.om.ratis.server.failure.timeout.duration=120s, ozone.om.ratis.server.leaderelection.pre-vote=true, ozone.om.ratis.server.request.timeout=3s, ozone.om.ratis.server.retry.cache.timeout=600000ms, ozone.om.ratis.snapshot.max.total.sst.size=100000000, ozone.om.read.threadpool=10, ozone.om.s3.grpc.server_enabled=true, ozone.om.save.metrics.interval=5m, ozone.om.security.admin.protocol.acl=*, ozone.om.security.client.protocol.acl=*, ozone.om.server.list.max.size=1000, ozone.om.service.ids=omservice, ozone.om.snapshot.cache.cleanup.service.run.interval=1m, ozone.om.snapshot.cache.max.size=10, ozone.om.snapshot.checkpoint.dir.creation.poll.timeout=20s, ozone.om.snapshot.compact.non.snapshot.diff.tables=false, ozone.om.snapshot.compaction.dag.max.time.allowed=30d, ozone.om.snapshot.compaction.dag.prune.daemon.run.interval=3600s, ozone.om.snapshot.db.max.open.files=100, ozone.om.snapshot.diff.cleanup.service.run.interval=1m, ozone.om.snapshot.diff.cleanup.service.timeout=5m, ozone.om.snapshot.diff.disable.native.libs=false, ozone.om.snapshot.diff.job.default.wait.time=1m, ozone.om.snapshot.diff.job.report.persistent.time=7d, ozone.om.snapshot.diff.max.allowed.keys.changed.per.job=10000000, ozone.om.snapshot.diff.max.jobs.purge.per.task=100, ozone.om.snapshot.diff.max.page.size=1000, ozone.om.snapshot.diff.thread.pool.size=10, ozone.om.snapshot.force.full.diff=false, ozone.om.snapshot.load.native.lib=true, ozone.om.snapshot.provider.connection.timeout=5000s, ozone.om.snapshot.provider.request.timeout=300000ms, ozone.om.snapshot.provider.socket.timeout=5000s, ozone.om.snapshot.prune.compaction.backup.batch.size=2000, ozone.om.snapshot.rocksdb.metrics.enabled=false, ozone.om.transport.class=org.apache.hadoop.ozone.om.protocolPB.Hadoop3OmTransportFactory, ozone.om.unflushed.transaction.max.count=10000, ozone.om.upgrade.finalization.ratis.based.timeout=30s, ozone.om.upgrade.quota.recalculate.enabled=true, ozone.om.user.max.volume=1024, ozone.om.user.rights=ALL, ozone.om.volume.listall.allowed=false, ozone.recon.address=recon:9891, ozone.recon.containerkey.flush.db.max.threshold=150000, ozone.recon.db.dir=/data/metadata/recon, ozone.recon.db.dirs.permissions=750, ozone.recon.heatmap.enable=false, ozone.recon.http-address=0.0.0.0:9888, ozone.recon.http-bind-host=0.0.0.0, ozone.recon.http.auth.kerberos.keytab=/etc/security/keytabs/recon.keytab, ozone.recon.http.auth.kerberos.principal=HTTP/recon@EXAMPLE.COM, ozone.recon.http.auth.type=kerberos, ozone.recon.http.enabled=true, ozone.recon.https-address=0.0.0.0:9889, ozone.recon.https-bind-host=0.0.0.0, ozone.recon.kerberos.keytab.file=/etc/security/keytabs/recon.keytab, ozone.recon.kerberos.principal=recon/recon@EXAMPLE.COM, ozone.recon.nssummary.flush.db.max.threshold=150000, ozone.recon.om.connection.request.timeout=5000, ozone.recon.om.connection.timeout=5s, ozone.recon.om.snapshot.task.flush.param=false, ozone.recon.om.snapshot.task.initial.delay=20s, ozone.recon.om.snapshot.task.interval.delay=1m, ozone.recon.om.socket.timeout=5s, ozone.recon.scm.connection.request.timeout=5s, ozone.recon.scm.connection.timeout=5s, ozone.recon.scm.container.threshold=100, ozone.recon.scm.snapshot.enabled=true, ozone.recon.scm.snapshot.task.initial.delay=1m, ozone.recon.scm.snapshot.task.interval.delay=24h, ozone.recon.scmclient.failover.max.retry=3, ozone.recon.scmclient.max.retry.timeout=6s, ozone.recon.scmclient.rpc.timeout=1m, ozone.recon.security.client.datanode.container.protocol.acl=*, ozone.recon.sql.db.auto.commit=true, ozone.recon.sql.db.conn.idle.max.age=3600s, ozone.recon.sql.db.conn.idle.test=SELECT 1, ozone.recon.sql.db.conn.idle.test.period=60s, ozone.recon.sql.db.conn.max.active=5, ozone.recon.sql.db.conn.max.age=1800s, ozone.recon.sql.db.conn.timeout=30000ms, ozone.recon.sql.db.driver=org.apache.derby.jdbc.EmbeddedDriver, ozone.recon.sql.db.jdbc.url=jdbc:derby:/data/metadata/recon/ozone_recon_derby.db, ozone.recon.sql.db.jooq.dialect=DERBY, ozone.recon.task.containercounttask.interval=60s, ozone.recon.task.missingcontainer.interval=300s, ozone.recon.task.pipelinesync.interval=300s, ozone.recon.task.safemode.wait.threshold=300s, ozone.recon.task.thread.count=1, ozone.replication.allowed-configs=^((STANDALONE|RATIS)/(ONE|THREE))|(EC/(3-2|6-3|10-4)-(512|1024|2048|4096)k)$, ozone.rest.client.http.connection.max=100, ozone.rest.client.http.connection.per-route.max=20, ozone.s3.administrators=testuser,s3g, ozone.s3g.client.buffer.size=4MB, ozone.s3g.default.bucket.layout=OBJECT_STORE, ozone.s3g.http-address=0.0.0.0:9878, ozone.s3g.http-bind-host=0.0.0.0, ozone.s3g.http.auth.kerberos.keytab=/etc/security/keytabs/HTTP.keytab, ozone.s3g.http.auth.kerberos.principal=HTTP/s3g@EXAMPLE.COM, ozone.s3g.http.auth.type=kerberos, ozone.s3g.http.enabled=true, ozone.s3g.https-address=0.0.0.0:9879, ozone.s3g.kerberos.keytab.file=/etc/security/keytabs/s3g.keytab, ozone.s3g.kerberos.principal=s3g/s3g@EXAMPLE.COM, ozone.s3g.list-keys.shallow.enabled=true, ozone.s3g.list.max.keys.limit=1000, ozone.s3g.metrics.percentiles.intervals.seconds=60, ozone.s3g.secret.http.auth.type=kerberos, ozone.s3g.secret.http.enabled=true, ozone.s3g.volume.name=s3v, ozone.s3g.webadmin.http-address=0.0.0.0:19878, ozone.s3g.webadmin.http-bind-host=0.0.0.0, ozone.s3g.webadmin.http.enabled=true, ozone.s3g.webadmin.https-address=0.0.0.0:19879, ozone.scm.address.scmservice.scm1=scm1.org, ozone.scm.address.scmservice.scm2=scm2.org, ozone.scm.address.scmservice.scm3=scm3.org, ozone.scm.block.client.address=scm, ozone.scm.block.client.bind.host=0.0.0.0, ozone.scm.block.client.port=9863, ozone.scm.block.deletion.max.retry=4096, ozone.scm.block.handler.count.key=100, ozone.scm.block.read.threadpool=10, ozone.scm.block.size=256MB, ozone.scm.ca.list.retry.interval=10s, ozone.scm.chunk.size=4MB, ozone.scm.client.address=scm, ozone.scm.client.bind.host=0.0.0.0, ozone.scm.client.handler.count.key=100, ozone.scm.client.port=9860, ozone.scm.client.read.threadpool=10, ozone.scm.close.container.wait.duration=5s, ozone.scm.container.layout=FILE_PER_BLOCK, ozone.scm.container.list.max.count=4096, ozone.scm.container.lock.stripes=512, ozone.scm.container.placement.ec.impl=org.apache.hadoop.hdds.scm.container.placement.algorithms.SCMContainerPlacementRackScatter, ozone.scm.container.placement.impl=org.apache.hadoop.hdds.scm.container.placement.algorithms.SCMContainerPlacementRackAware, ozone.scm.container.size=1GB, ozone.scm.datanode.admin.monitor.interval=30s, ozone.scm.datanode.admin.monitor.logging.limit=1000, ozone.scm.datanode.disallow.same.peers=false, ozone.scm.datanode.handler.count.key=100, ozone.scm.datanode.id.dir=/data/metadata, ozone.scm.datanode.pipeline.limit=2, ozone.scm.datanode.port=9861, ozone.scm.datanode.ratis.volume.free-space.min=10MB, ozone.scm.datanode.read.threadpool=10, ozone.scm.db.dirs.permissions=750, ozone.scm.dead.node.interval=10m, ozone.scm.ec.pipeline.minimum=5, ozone.scm.ec.pipeline.per.volume.factor=1, ozone.scm.event.ContainerReport.thread.pool.size=10, ozone.scm.expired.container.replica.op.scrub.interval=5m, ozone.scm.grpc.port=9895, ozone.scm.ha.dbtransactionbuffer.flush.interval=600s, ozone.scm.ha.grpc.deadline.interval=30m, ozone.scm.ha.raft.server.log.appender.wait-time.min=0ms, ozone.scm.ha.ratis.leader.election.timeout=5s, ozone.scm.ha.ratis.leader.ready.check.interval=2s, ozone.scm.ha.ratis.leader.ready.wait.timeout=60s, ozone.scm.ha.ratis.log.appender.queue.byte-limit=32MB, ozone.scm.ha.ratis.log.appender.queue.num-elements=1024, ozone.scm.ha.ratis.log.purge.enabled=false, ozone.scm.ha.ratis.log.purge.gap=1000000, ozone.scm.ha.ratis.request.timeout=30s, ozone.scm.ha.ratis.rpc.type=GRPC, ozone.scm.ha.ratis.segment.preallocated.size=4MB, ozone.scm.ha.ratis.segment.size=4MB, ozone.scm.ha.ratis.server.failure.timeout.duration=120s, ozone.scm.ha.ratis.server.leaderelection.pre-vote=true, ozone.scm.ha.ratis.server.retry.cache.timeout=60s, ozone.scm.ha.ratis.server.snapshot.creation.gap=1024, ozone.scm.ha.ratis.snapshot.threshold=1000, ozone.scm.handler.count.key=100, ozone.scm.heartbeat.log.warn.interval.count=10, ozone.scm.heartbeat.rpc-retry-count=15, ozone.scm.heartbeat.rpc-retry-interval=1s, ozone.scm.heartbeat.rpc-timeout=5s, ozone.scm.heartbeat.thread.interval=3s, ozone.scm.http-address=0.0.0.0:9876, ozone.scm.http-bind-host=0.0.0.0, ozone.scm.http.enabled=true, ozone.scm.https-address=0.0.0.0:9877, ozone.scm.https-bind-host=0.0.0.0, ozone.scm.info.wait.duration=10m, ozone.scm.keyvalue.container.deletion-choosing.policy=org.apache.hadoop.ozone.container.common.impl.TopNOrderedContainerDeletionChoosingPolicy, ozone.scm.network.topology.schema.file=network-topology-default.xml, ozone.scm.nodes.scmservice=scm1,scm2,scm3, ozone.scm.pipeline.allocated.timeout=5m, ozone.scm.pipeline.creation.auto.factor.one=true, ozone.scm.pipeline.creation.interval=30s, ozone.scm.pipeline.destroy.timeout=66s, ozone.scm.pipeline.leader-choose.policy=org.apache.hadoop.hdds.scm.pipeline.leader.choose.algorithms.MinLeaderCountChoosePolicy, ozone.scm.pipeline.owner.container.count=1, ozone.scm.pipeline.per.metadata.disk=2, ozone.scm.pipeline.scrub.interval=5m, ozone.scm.primordial.node.id=scm1, ozone.scm.ratis.pipeline.limit=0, ozone.scm.ratis.port=9894, ozone.scm.security.handler.count.key=2, ozone.scm.security.read.threadpool=1, ozone.scm.security.service.bind.host=0.0.0.0, ozone.scm.security.service.port=9961, ozone.scm.sequence.id.batch.size=1000, ozone.scm.service.ids=scmservice, ozone.scm.skip.bootstrap.validation=false, ozone.scm.stale.node.interval=5m, ozone.security.crypto.compliance.mode=unrestricted, ozone.security.enabled=true, ozone.security.http.kerberos.enabled=true, ozone.security.reconfigure.protocol.acl=*, ozone.server.default.replication=3, ozone.server.default.replication.type=RATIS, ozone.service.shutdown.timeout=60s, ozone.snapshot.deep.cleaning.enabled=false, ozone.snapshot.deleting.limit.per.task=10, ozone.snapshot.deleting.service.interval=30s, ozone.snapshot.deleting.service.timeout=300s, ozone.snapshot.directory.service.interval=24h, ozone.snapshot.directory.service.timeout=300s, ozone.snapshot.filtering.limit.per.task=2, ozone.snapshot.filtering.service.interval=1m, ozone.snapshot.key.deleting.limit.per.task=20000, ozone.sst.filtering.service.timeout=300000ms, ozone.volume.io.percentiles.intervals.seconds=60, ozone.xceiver.client.metrics.percentiles.intervals.seconds=60, recon.om.delta.update.lag.threshold=0, recon.om.delta.update.limit=50000, scm.container.client.idle.threshold=10s, scm.container.client.max.size=256}
************************************************************/
2025-06-17 21:40:47,443 [main] INFO recon.ReconServer: registered UNIX signal handlers for [SIGHUP, SIGINT, SIGTERM]
2025-06-17 21:40:49,230 [main] INFO reflections.Reflections: Reflections took 300 ms to scan 1 urls, producing 21 keys and 84 values
2025-06-17 21:40:53,618 [main] INFO reflections.Reflections: Reflections took 241 ms to scan 3 urls, producing 128 keys and 286 values
2025-06-17 21:40:53,819 [main] INFO recon.ReconServer: Initializing Recon server...
2025-06-17 21:40:54,067 [main] INFO recon.ReconServer: Ozone security is enabled. Attempting login for Recon service. Principal: recon/recon@EXAMPLE.COM, keytab: /etc/security/keytabs/recon.keytab
2025-06-17 21:40:54,391 [main] INFO security.UserGroupInformation: Login successful for user recon/recon@EXAMPLE.COM using keytab file recon.keytab. Keytab auto renewal enabled : false
2025-06-17 21:40:54,391 [main] INFO recon.ReconServer: Recon login successful.
2025-06-17 21:40:54,448 [main] INFO recon.ReconServer: ReconStorageConfig initialized.Initializing certificate.
2025-06-17 21:40:54,448 [main] INFO recon.ReconServer: Initializing secure Recon.
2025-06-17 21:40:54,518 [main] INFO proxy.SCMSecurityProtocolFailoverProxyProvider: Created fail-over proxy for protocol SCMSecurityProtocolPB with 3 nodes: [nodeId=scm2,nodeAddress=scm2.org/172.25.0.117:9961, nodeId=scm1,nodeAddress=scm1.org/172.25.0.116:9961, nodeId=scm3,nodeAddress=scm3.org/172.25.0.118:9961]
2025-06-17 21:40:54,631 [main] INFO security.ReconCertificateClient: Certificate serial ID set to null
2025-06-17 21:40:54,634 [main] ERROR security.ReconCertificateClient: Default certificate serial id is not set. Can't locate the default certificate for this client.
2025-06-17 21:40:54,634 [main] INFO security.ReconCertificateClient: Certificate client init case: 0
2025-06-17 21:40:54,637 [main] INFO security.ReconCertificateClient: Creating keypair for client as keypair and certificate not found.
2025-06-17 21:40:55,757 [main] INFO keys.KeyStorage: Storing public key to /data/metadata/recon/keys/public.pem.
2025-06-17 21:40:55,763 [main] INFO keys.KeyStorage: Storing private key to /data/metadata/recon/keys/private.pem.
2025-06-17 21:40:55,771 [main] INFO security.ReconCertificateClient: Init response: GETCERT
2025-06-17 21:40:55,772 [main] INFO security.ReconCertificateClient: Creating CSR for Recon.
2025-06-17 21:40:55,789 [main] INFO ozone.OzoneSecurityUtil: Adding ip:172.25.0.115,host:recon
2025-06-17 21:40:55,789 [main] INFO ozone.OzoneSecurityUtil: ip:0:0:0:0:0:0:0:1%lo not returned.
2025-06-17 21:40:55,790 [main] INFO ozone.OzoneSecurityUtil: ip:127.0.0.1 not returned.
2025-06-17 21:40:55,822 [main] ERROR utils.CertificateSignRequest: Invalid domain recon
2025-06-17 21:41:02,202 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm1.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm1,nodeAddress=scm1.org/172.25.0.116:9961 after 3 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 3.
2025-06-17 21:41:04,204 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm2.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm2,nodeAddress=scm2.org/172.25.0.117:9961 after 4 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 4.
2025-06-17 21:41:06,205 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm3.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm3,nodeAddress=scm3.org/172.25.0.118:9961 after 5 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 5.
2025-06-17 21:41:08,209 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm1.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm1,nodeAddress=scm1.org/172.25.0.116:9961 after 6 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 6.
2025-06-17 21:41:10,212 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm2.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm2,nodeAddress=scm2.org/172.25.0.117:9961 after 7 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 7.
2025-06-17 21:41:12,214 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm3.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm3,nodeAddress=scm3.org/172.25.0.118:9961 after 8 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 8.
2025-06-17 21:41:14,378 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.hdds.ratis.ServerNotLeaderException): SCM Server:c1ee1959-3a78-4ea6-8a15-7cb51d6dd9c3(scm1.org) is not the leader. Could not determine the leader node.
	at org.apache.hadoop.hdds.ratis.ServerNotLeaderException.convertToNotLeaderException(ServerNotLeaderException.java:106)
	at org.apache.hadoop.hdds.scm.ha.RatisUtil.checkRatisException(RatisUtil.java:248)
	at org.apache.hadoop.hdds.scm.protocol.SCMSecurityProtocolServerSideTranslatorPB.submitRequest(SCMSecurityProtocolServerSideTranslatorPB.java:83)
	at org.apache.hadoop.hdds.protocol.proto.SCMSecurityProtocolProtos$SCMSecurityProtocolService$2.callBlockingMethod(SCMSecurityProtocolProtos.java:18942)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy52.submitRequest over nodeId=scm1,nodeAddress=scm1.org/172.25.0.116:9961 after 9 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 9.
2025-06-17 21:41:16,381 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm2.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm2,nodeAddress=scm2.org/172.25.0.117:9961 after 10 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 10.
2025-06-17 21:41:18,383 [main] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to scm3.org:9961 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy52.submitRequest over nodeId=scm3,nodeAddress=scm3.org/172.25.0.118:9961 after 11 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 11.
2025-06-17 21:41:20,833 [main] INFO utils.CertificateCodec: Save certificate to /data/metadata/recon/certs/4.crt
2025-06-17 21:41:20,834 [main] INFO utils.CertificateCodec: Certificate -----BEGIN CERTIFICATE-----
MIIDrTCCApWgAwIBAgIBBDANBgkqhkiG9w0BAQsFADCBiTEZMBcGA1UEAwwQc2Nt
LXN1YkBzY20xLm9yZzEtMCsGA1UECwwkYzFlZTE5NTktM2E3OC00ZWE2LThhMTUt
N2NiNTFkNmRkOWMzMTEwLwYDVQQKDChDSUQtNmY0ZGM3ZTAtMzViYi00ZTE2LWE2
YjgtM2NjMTRhY2FmMGFlMQowCAYDVQQFEwEyMB4XDTI1MDYxNzIxNDEyMFoXDTI2
MDYxNzIxNDEyMFowgYQxFDASBgNVBAMMC3JlY29uQHJlY29uMS0wKwYDVQQLDCRj
MWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJ
RC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0zY2MxNGFjYWYwYWUxCjAIBgNVBAUT
ATQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCoKjLJYyt0qz8BUm5A
silns9emBScIJRbXJP7+Ls0pXhHFhSm+qKgCQaKmYzdM3L/YBT4iTiaIYTRffYUE
cBaShb/cB+1cbk01ejmtrazUbG1998EWUapggVTjRnFMKD7+BKJgyL3dQo2ctD4J
Y7ypiXmUkwJwxu9iX+BTyXOvxjyTZp3kapiZJdaED8qrLghs3SfOI8qsq79LAiA5
t6d2ROjqTXk9wBFkpUvQcYHsuv0KasB6nIbAYSKxZ22yHA4UaTt67JF4kxfps5/o
uDTyXuxy1r2AXEHQ3XYJud0pdY8ffpbsXSOQmIdpPV5udN1KotjKriF86amkJwgh
3jarAgMBAAGjIzAhMA4GA1UdDwEB/wQEAwIDuDAPBgNVHREECDAGhwSsGQBzMA0G
CSqGSIb3DQEBCwUAA4IBAQAHAUWubk7IbHp5e8HvpNMrI2wZ+k1vGcQrGPBWCdXl
F/iBxrwsVAV5X+/LT3pUyG8sFdJJOizUb2X++KnxZwy4WPkL84ZdGgmiDPz0OuI6
MyO1+tbJ9vNAF4zPds5/kzQXWjx9kt3k71haX+BXVPjfmvXfMDtT5fLAqRLx9FZR
4oYJFukFkwjW2ghhpb75w/7pGkZOWbfM2NgqNSnLLBo2hqvzo+o+y3GxB47N5xeg
GyTkFBs+YBoGT2q1QEvWk8ufbvqzMx92HK8HJeG34gk1bIoB7peR6cTX5yKcog1O
3YQqHX+cPzDuVfc2MuEyqS7fjBci3/QJXr4ITLIUaYhS
-----END CERTIFICATE-----

-----BEGIN CERTIFICATE-----
MIIDyTCCArGgAwIBAgIBAjANBgkqhkiG9w0BAQsFADCBhTEVMBMGA1UEAwwMc2Nt
QHNjbTEub3JnMS0wKwYDVQQLDCRjMWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1
MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0z
Y2MxNGFjYWYwYWUxCjAIBgNVBAUTATEwHhcNMjUwNjE3MjE0MDUzWhcNMzAwNzI2
MjE0MDUzWjCBiTEZMBcGA1UEAwwQc2NtLXN1YkBzY20xLm9yZzEtMCsGA1UECwwk
YzFlZTE5NTktM2E3OC00ZWE2LThhMTUtN2NiNTFkNmRkOWMzMTEwLwYDVQQKDChD
SUQtNmY0ZGM3ZTAtMzViYi00ZTE2LWE2YjgtM2NjMTRhY2FmMGFlMQowCAYDVQQF
EwEyMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvxWmTnCX2wMU166h
V1vcV3rHvyK6ws6B3Ktwe+4BoUEDpSbqzmr5LH5+S1A9OX5MS7T8uMLi6tzMwfj8
trBD70t4Daza7BvBGCs8Rp/bVLu4XsVF29KkJ8iNtU4pj6BIZYG0K0cQDd6qYbhF
FGNFHPX0BmR0vIk3Qy7jykpZp74eBLM1BNhWyY1JZ8w5q8CV8TDKLnIkN/z13gVQ
dWWZC4woETX5AK/wcYdtubLJTP7szJqbsrOVXjdXFFYqdRGIEu01NWYmpQX8njAr
9KUfBSwi2e1SYBzvT7SM2ha1VO0ZL6ZgLNLJxhPBwNzgAoy8Ok9mLIM8cYU1BDF8
aFotuQIDAQABoz4wPDAOBgNVHQ8BAf8EBAMCAb4wGQYDVR0RBBIwEIcErBkAdIII
c2NtMS5vcmcwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAtFRs
XyuB2rKDQ98Bh7mJWK73CrNrhbIMNvm/Z/1Gb2ckgLcGlUZ5OC5vlA+OSqiTFizV
Ax6WllUFhCphgELhD3mNn0OEob+onn3Qur54O5t23obztDoz4FLswPAoV+Ox7ltm
bMSY7SyhEv9rSWf1P6TOdiQ/GU/l4zdF8QBDE+qtMcdvhznF3AGUi2vq2XcwXwnO
JHEUiIIFcEvfsgfQj7FGTkAtj4Wk/iGcf3ods3iYkBA8Ie1J7FPd3jLWBz6tCYev
+IfNTHWzP06WE0cVtHFwgSyodfYX62YqjV24u0GeqC0XVfjYUmneLyFl/HisfvUX
rNYyen+y1d3rH9nxSQ==
-----END CERTIFICATE-----

-----BEGIN CERTIFICATE-----
MIIDxTCCAq2gAwIBAgIBATANBgkqhkiG9w0BAQsFADCBhTEVMBMGA1UEAwwMc2Nt
QHNjbTEub3JnMS0wKwYDVQQLDCRjMWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1
MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0z
Y2MxNGFjYWYwYWUxCjAIBgNVBAUTATEwHhcNMjUwNjE3MjE0MDUzWhcNMzAwNzI2
MjE0MDUzWjCBhTEVMBMGA1UEAwwMc2NtQHNjbTEub3JnMS0wKwYDVQQLDCRjMWVl
MTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02
ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0zY2MxNGFjYWYwYWUxCjAIBgNVBAUTATEw
ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDGeV2ytUGyytQYuixdWw23
DgfHpmInDFcqCe+KhcfJIJoFyYl7Yc4nE/UlnYMq4UkAHXqoh30XCYi5gflgfMlo
pg0R44gQuG/J0lreEWC4n2F3KiLbR+yQrAPQb9tgpUMtT44ZmNhQ4H33YpXNuBpc
6vJgab5goDUNQ2r8oEiVeE1s786s4L8WTlzZErepua9qkorB0v9JgL2NBC3536MZ
fhuhcbacRY40a4rX/v2NGRltODeE5vxe4ihEoieM/BeaL+jcT2ZKtOgmG8gpMpai
v+zgAYbyq5vIkg7UEHjse/p6oCNAbRmAetm8cJsbatNKU9lNmy8poMqHJCSu2JO5
AgMBAAGjPjA8MA8GA1UdEwEB/wQFMAMBAf8wDgYDVR0PAQH/BAQDAgEGMBkGA1Ud
EQQSMBCHBKwZAHSCCHNjbTEub3JnMA0GCSqGSIb3DQEBCwUAA4IBAQBG83AcK9Dv
WVYlTpbDe7rDT/W0jqg/+Ni70vgRH3gYcm2hTffO9NPuFO8F9r8JgyM7zQUAkLiX
J/KI0tfj+A8/9oZqZ0wOCu/XhnYUNwVYT/ESXecyFCuQhBIZAIi1CapZK1UKVwhL
EezgbecyvrikaSnBCZXG1XR/Mm9PztNMStEFbE86D8BxryZ+N86Bx8a7cnQr15Oh
vxssNfnXKrmlxQO/ukcuGprVIJ45vnWDAE8kGXH5KhM1YXI0j9JO9QZtA7VGipWW
OHKtelltyThFe6sFXZ2lV/LyELIQh5jFdHVAzWQcjJ6RFXlVuHCRFXvTkIrZh9a7
BCPNlhqZoSv2
-----END CERTIFICATE-----

2025-06-17 21:41:20,847 [main] INFO utils.CertificateCodec: Save certificate to /data/metadata/recon/certs/CA-2.crt
2025-06-17 21:41:20,847 [main] INFO utils.CertificateCodec: Certificate -----BEGIN CERTIFICATE-----
MIIDyTCCArGgAwIBAgIBAjANBgkqhkiG9w0BAQsFADCBhTEVMBMGA1UEAwwMc2Nt
QHNjbTEub3JnMS0wKwYDVQQLDCRjMWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1
MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0z
Y2MxNGFjYWYwYWUxCjAIBgNVBAUTATEwHhcNMjUwNjE3MjE0MDUzWhcNMzAwNzI2
MjE0MDUzWjCBiTEZMBcGA1UEAwwQc2NtLXN1YkBzY20xLm9yZzEtMCsGA1UECwwk
YzFlZTE5NTktM2E3OC00ZWE2LThhMTUtN2NiNTFkNmRkOWMzMTEwLwYDVQQKDChD
SUQtNmY0ZGM3ZTAtMzViYi00ZTE2LWE2YjgtM2NjMTRhY2FmMGFlMQowCAYDVQQF
EwEyMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvxWmTnCX2wMU166h
V1vcV3rHvyK6ws6B3Ktwe+4BoUEDpSbqzmr5LH5+S1A9OX5MS7T8uMLi6tzMwfj8
trBD70t4Daza7BvBGCs8Rp/bVLu4XsVF29KkJ8iNtU4pj6BIZYG0K0cQDd6qYbhF
FGNFHPX0BmR0vIk3Qy7jykpZp74eBLM1BNhWyY1JZ8w5q8CV8TDKLnIkN/z13gVQ
dWWZC4woETX5AK/wcYdtubLJTP7szJqbsrOVXjdXFFYqdRGIEu01NWYmpQX8njAr
9KUfBSwi2e1SYBzvT7SM2ha1VO0ZL6ZgLNLJxhPBwNzgAoy8Ok9mLIM8cYU1BDF8
aFotuQIDAQABoz4wPDAOBgNVHQ8BAf8EBAMCAb4wGQYDVR0RBBIwEIcErBkAdIII
c2NtMS5vcmcwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAtFRs
XyuB2rKDQ98Bh7mJWK73CrNrhbIMNvm/Z/1Gb2ckgLcGlUZ5OC5vlA+OSqiTFizV
Ax6WllUFhCphgELhD3mNn0OEob+onn3Qur54O5t23obztDoz4FLswPAoV+Ox7ltm
bMSY7SyhEv9rSWf1P6TOdiQ/GU/l4zdF8QBDE+qtMcdvhznF3AGUi2vq2XcwXwnO
JHEUiIIFcEvfsgfQj7FGTkAtj4Wk/iGcf3ods3iYkBA8Ie1J7FPd3jLWBz6tCYev
+IfNTHWzP06WE0cVtHFwgSyodfYX62YqjV24u0GeqC0XVfjYUmneLyFl/HisfvUX
rNYyen+y1d3rH9nxSQ==
-----END CERTIFICATE-----

-----BEGIN CERTIFICATE-----
MIIDxTCCAq2gAwIBAgIBATANBgkqhkiG9w0BAQsFADCBhTEVMBMGA1UEAwwMc2Nt
QHNjbTEub3JnMS0wKwYDVQQLDCRjMWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1
MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0z
Y2MxNGFjYWYwYWUxCjAIBgNVBAUTATEwHhcNMjUwNjE3MjE0MDUzWhcNMzAwNzI2
MjE0MDUzWjCBhTEVMBMGA1UEAwwMc2NtQHNjbTEub3JnMS0wKwYDVQQLDCRjMWVl
MTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02
ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0zY2MxNGFjYWYwYWUxCjAIBgNVBAUTATEw
ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDGeV2ytUGyytQYuixdWw23
DgfHpmInDFcqCe+KhcfJIJoFyYl7Yc4nE/UlnYMq4UkAHXqoh30XCYi5gflgfMlo
pg0R44gQuG/J0lreEWC4n2F3KiLbR+yQrAPQb9tgpUMtT44ZmNhQ4H33YpXNuBpc
6vJgab5goDUNQ2r8oEiVeE1s786s4L8WTlzZErepua9qkorB0v9JgL2NBC3536MZ
fhuhcbacRY40a4rX/v2NGRltODeE5vxe4ihEoieM/BeaL+jcT2ZKtOgmG8gpMpai
v+zgAYbyq5vIkg7UEHjse/p6oCNAbRmAetm8cJsbatNKU9lNmy8poMqHJCSu2JO5
AgMBAAGjPjA8MA8GA1UdEwEB/wQFMAMBAf8wDgYDVR0PAQH/BAQDAgEGMBkGA1Ud
EQQSMBCHBKwZAHSCCHNjbTEub3JnMA0GCSqGSIb3DQEBCwUAA4IBAQBG83AcK9Dv
WVYlTpbDe7rDT/W0jqg/+Ni70vgRH3gYcm2hTffO9NPuFO8F9r8JgyM7zQUAkLiX
J/KI0tfj+A8/9oZqZ0wOCu/XhnYUNwVYT/ESXecyFCuQhBIZAIi1CapZK1UKVwhL
EezgbecyvrikaSnBCZXG1XR/Mm9PztNMStEFbE86D8BxryZ+N86Bx8a7cnQr15Oh
vxssNfnXKrmlxQO/ukcuGprVIJ45vnWDAE8kGXH5KhM1YXI0j9JO9QZtA7VGipWW
OHKtelltyThFe6sFXZ2lV/LyELIQh5jFdHVAzWQcjJ6RFXlVuHCRFXvTkIrZh9a7
BCPNlhqZoSv2
-----END CERTIFICATE-----

2025-06-17 21:41:20,859 [main] INFO utils.CertificateCodec: Save certificate to /data/metadata/recon/certs/ROOTCA-1.crt
2025-06-17 21:41:20,859 [main] INFO utils.CertificateCodec: Certificate -----BEGIN CERTIFICATE-----
MIIDxTCCAq2gAwIBAgIBATANBgkqhkiG9w0BAQsFADCBhTEVMBMGA1UEAwwMc2Nt
QHNjbTEub3JnMS0wKwYDVQQLDCRjMWVlMTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1
MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0z
Y2MxNGFjYWYwYWUxCjAIBgNVBAUTATEwHhcNMjUwNjE3MjE0MDUzWhcNMzAwNzI2
MjE0MDUzWjCBhTEVMBMGA1UEAwwMc2NtQHNjbTEub3JnMS0wKwYDVQQLDCRjMWVl
MTk1OS0zYTc4LTRlYTYtOGExNS03Y2I1MWQ2ZGQ5YzMxMTAvBgNVBAoMKENJRC02
ZjRkYzdlMC0zNWJiLTRlMTYtYTZiOC0zY2MxNGFjYWYwYWUxCjAIBgNVBAUTATEw
ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDGeV2ytUGyytQYuixdWw23
DgfHpmInDFcqCe+KhcfJIJoFyYl7Yc4nE/UlnYMq4UkAHXqoh30XCYi5gflgfMlo
pg0R44gQuG/J0lreEWC4n2F3KiLbR+yQrAPQb9tgpUMtT44ZmNhQ4H33YpXNuBpc
6vJgab5goDUNQ2r8oEiVeE1s786s4L8WTlzZErepua9qkorB0v9JgL2NBC3536MZ
fhuhcbacRY40a4rX/v2NGRltODeE5vxe4ihEoieM/BeaL+jcT2ZKtOgmG8gpMpai
v+zgAYbyq5vIkg7UEHjse/p6oCNAbRmAetm8cJsbatNKU9lNmy8poMqHJCSu2JO5
AgMBAAGjPjA8MA8GA1UdEwEB/wQFMAMBAf8wDgYDVR0PAQH/BAQDAgEGMBkGA1Ud
EQQSMBCHBKwZAHSCCHNjbTEub3JnMA0GCSqGSIb3DQEBCwUAA4IBAQBG83AcK9Dv
WVYlTpbDe7rDT/W0jqg/+Ni70vgRH3gYcm2hTffO9NPuFO8F9r8JgyM7zQUAkLiX
J/KI0tfj+A8/9oZqZ0wOCu/XhnYUNwVYT/ESXecyFCuQhBIZAIi1CapZK1UKVwhL
EezgbecyvrikaSnBCZXG1XR/Mm9PztNMStEFbE86D8BxryZ+N86Bx8a7cnQr15Oh
vxssNfnXKrmlxQO/ukcuGprVIJ45vnWDAE8kGXH5KhM1YXI0j9JO9QZtA7VGipWW
OHKtelltyThFe6sFXZ2lV/LyELIQh5jFdHVAzWQcjJ6RFXlVuHCRFXvTkIrZh9a7
BCPNlhqZoSv2
-----END CERTIFICATE-----

2025-06-17 21:41:20,862 [main] INFO security.ReconCertificateClient: Certificate serial ID set to 4
2025-06-17 21:41:20,872 [main] INFO security.ReconCertificateClient: Added certificate 1 from file: /data/metadata/recon/certs/ROOTCA-1.crt.
2025-06-17 21:41:20,883 [main] INFO security.ReconCertificateClient: Added certificate 2 from file: /data/metadata/recon/certs/CA-2.crt.
2025-06-17 21:41:20,894 [main] INFO security.ReconCertificateClient: Added certificate 4 from file: /data/metadata/recon/certs/4.crt.
2025-06-17 21:41:20,921 [main] INFO security.ReconCertificateClient: CertificateRenewerService for recon is started with first delay 29116799096 ms and interval 86400000 ms.
2025-06-17 21:41:20,937 [main] INFO security.ReconCertificateClient: Successfully stored RECON signed certificate, case:GETCERT.
2025-06-17 21:41:21,552 [main] INFO persistence.DefaultDataSourceProvider: JDBC Url for Recon : jdbc:derby:/data/metadata/recon/ozone_recon_derby.db 
2025-06-17 21:41:23,416 [main] INFO schema.SqlDbUtils: Created derby database at jdbc:derby:/data/metadata/recon/ozone_recon_derby.db.
2025-06-17 21:41:23,989 [main] INFO impl.ReconContainerMetadataManagerImpl: KEY_CONTAINER Table is empty, initializing from CONTAINER_KEY Table ...
2025-06-17 21:41:23,994 [main] INFO impl.ReconContainerMetadataManagerImpl: It took 0.003 seconds to initialized 0 records to KEY_CONTAINER table
2025-06-17 21:41:24,112 [main] INFO persistence.DefaultDataSourceProvider: JDBC Url for Recon : jdbc:derby:/data/metadata/recon/ozone_recon_derby.db 
2025-06-17 21:41:24,137 [main] INFO schema.SqlDbUtils: Created derby database at jdbc:derby:/data/metadata/recon/ozone_recon_derby.db.
2025-06-17 21:41:24,138 [main] INFO recon.ReconServer: Creating Recon Schema.
2025-06-17 21:41:25,106 [main] INFO schema.SchemaVersionTableDefinition: Inserted initial SLV '1' into SchemaVersion table.
2025-06-17 21:41:25,208 [main] INFO schema.ContainerSchemaDefinition: UNHEALTHY_CONTAINERS is missing creating new one.
2025-06-17 21:41:25,378 [main] INFO recon.ReconServer: Finalizing Layout Features.
2025-06-17 21:41:25,457 [main] INFO reflections.Reflections: Reflections took 5 ms to scan 1 urls, producing 6 keys and 9 values
2025-06-17 21:41:25,472 [main] INFO upgrade.ReconLayoutVersionManager: Current MLV: 1. SLV: 1. Checking features for registration...
2025-06-17 21:41:25,480 [main] INFO recon.ReconServer: Recon schema versioning completed.
2025-06-17 21:41:25,523 [main] INFO http.BaseHttpServer: Starting Web-server for recon at: http://0.0.0.0:9888
2025-06-17 21:41:25,523 [main] INFO http.BaseHttpServer: Hadoop Security Enabled: true Ozone Security Enabled: true Ozone HTTP Security Enabled: true 
2025-06-17 21:41:25,524 [main] INFO http.BaseHttpServer: HttpAuthType: ozone.recon.http.auth.type = kerberos
2025-06-17 21:41:25,551 [main] INFO util.log: Logging initialized @42615ms to org.eclipse.jetty.util.log.Slf4jLog
2025-06-17 21:41:25,832 [main] INFO http.HttpServer2: Added global filter 'safety' (class=org.apache.hadoop.hdds.server.http.HttpServer2$QuotingInputFilter)
2025-06-17 21:41:25,841 [main] INFO http.HttpServer2: Added filter authentication (class=org.apache.hadoop.security.authentication.server.AuthenticationFilter) to context recon
2025-06-17 21:41:25,841 [main] INFO http.HttpServer2: Added filter authentication (class=org.apache.hadoop.security.authentication.server.AuthenticationFilter) to context static
2025-06-17 21:41:25,843 [main] INFO http.HttpServer2: Added filter authentication (class=org.apache.hadoop.security.authentication.server.AuthenticationFilter) to context logs
2025-06-17 21:41:25,850 [main] INFO http.HttpServer2: Initialize spnego with host: 0.0.0.0 userKey: ozone.recon.http.auth.kerberos.principal keytabKey: ozone.recon.http.auth.kerberos.keytab
2025-06-17 21:41:25,917 [main] INFO http.BaseHttpServer: HTTP server of recon uses base directory /tmp/ozone_http
2025-06-17 21:41:25,953 [main] INFO tasks.ReconTaskControllerImpl: Registered task ContainerKeyMapperTaskFSO with controller.
2025-06-17 21:41:25,954 [main] INFO tasks.ReconTaskControllerImpl: Registered task ContainerKeyMapperTaskOBS with controller.
2025-06-17 21:41:25,954 [main] INFO tasks.ReconTaskControllerImpl: Registered task FileSizeCountTaskFSO with controller.
2025-06-17 21:41:25,954 [main] INFO tasks.ReconTaskControllerImpl: Registered task FileSizeCountTaskOBS with controller.
2025-06-17 21:41:25,954 [main] INFO tasks.ReconTaskControllerImpl: Registered task OmTableInsightTask with controller.
2025-06-17 21:41:25,954 [main] INFO tasks.ReconTaskControllerImpl: Registered task NSSummaryTask with controller.
2025-06-17 21:41:25,968 [main] INFO ozone.OmUtils: Using OzoneManager ServiceID 'omservice'.
2025-06-17 21:41:26,699 [main] WARN recon.ReconUtils: ozone.recon.om.db.dir is not configured. We recommend adding this setting. Falling back to ozone.metadata.dirs instead.
2025-06-17 21:41:26,749 [main] INFO proxy.SCMContainerLocationFailoverProxyProvider: Created fail-over proxy for protocol StorageContainerLocationProtocolPB with 3 nodes: [nodeId=scm2,nodeAddress=scm2.org/172.25.0.117:9860, nodeId=scm1,nodeAddress=scm1.org/172.25.0.116:9860, nodeId=scm3,nodeAddress=scm3.org/172.25.0.118:9860]
2025-06-17 21:41:26,821 [main] WARN recon.ReconUtils: ozone.recon.scm.db.dirs is not configured. We recommend adding this setting. Falling back to ozone.metadata.dirs instead.
2025-06-17 21:41:26,924 [main] INFO net.NodeSchemaLoader: Loading schema from [file:/etc/hadoop/network-topology-default.xml, jar:file:/opt/hadoop/share/ozone/lib/hdds-common-2.1.0-SNAPSHOT.jar!/network-topology-default.xml]
2025-06-17 21:41:26,926 [main] INFO net.NodeSchemaLoader: Loading network topology layer schema file
2025-06-17 21:41:27,110 [main] WARN db.DBStoreBuilder: ozone.recon.scm.db.dirs is not configured. We recommend adding this setting. Falling back to ozone.metadata.dirs instead.
2025-06-17 21:41:27,299 [main] INFO upgrade.AbstractLayoutVersionManager: Initializing Layout version manager with metadata layout = HBASE_SUPPORT (version = 8), software layout = HBASE_SUPPORT (version = 8)
2025-06-17 21:41:27,353 [main] INFO ha.SequenceIdGenerator: Init the HA SequenceIdGenerator.
2025-06-17 21:41:27,413 [main] INFO node.SCMNodeManager: Entering startup safe mode.
2025-06-17 21:41:27,424 [main] INFO scm.ReconNodeManager: Loaded 0 nodes from node DB.
2025-06-17 21:41:27,450 [main] INFO algorithms.ContainerPlacementPolicyFactory: Create container placement policy of type org.apache.hadoop.hdds.scm.container.placement.algorithms.SCMContainerPlacementRackAware
2025-06-17 21:41:28,269 [main] INFO audit.AuditLogger: Refresh DebugCmdSet for SCMAudit to [].
2025-06-17 21:41:28,336 [main] INFO ipc.CallQueueManager: Using callQueue: class java.util.concurrent.LinkedBlockingQueue, queueCapacity: 10000, scheduler: class org.apache.hadoop.ipc.DefaultRpcScheduler, ipcBackoff: false, ipcFailOver: false.
2025-06-17 21:41:28,340 [main] INFO ipc.Server: Listener at 0.0.0.0:9891
2025-06-17 21:41:28,341 [Socket Reader #1 for port 9891] INFO ipc.Server: Starting Socket Reader #1 for port 9891
2025-06-17 21:41:28,346 [Socket Reader #2 for port 9891] INFO ipc.Server: Starting Socket Reader #2 for port 9891
2025-06-17 21:41:28,347 [Socket Reader #3 for port 9891] INFO ipc.Server: Starting Socket Reader #3 for port 9891
2025-06-17 21:41:28,347 [Socket Reader #4 for port 9891] INFO ipc.Server: Starting Socket Reader #4 for port 9891
2025-06-17 21:41:28,347 [Socket Reader #5 for port 9891] INFO ipc.Server: Starting Socket Reader #5 for port 9891
2025-06-17 21:41:28,349 [Socket Reader #6 for port 9891] INFO ipc.Server: Starting Socket Reader #6 for port 9891
2025-06-17 21:41:28,349 [Socket Reader #7 for port 9891] INFO ipc.Server: Starting Socket Reader #7 for port 9891
2025-06-17 21:41:28,349 [Socket Reader #8 for port 9891] INFO ipc.Server: Starting Socket Reader #8 for port 9891
2025-06-17 21:41:28,350 [Socket Reader #9 for port 9891] INFO ipc.Server: Starting Socket Reader #9 for port 9891
2025-06-17 21:41:28,352 [Socket Reader #10 for port 9891] INFO ipc.Server: Starting Socket Reader #10 for port 9891
2025-06-17 21:41:28,423 [main] INFO pipeline.PipelineStateManagerImpl: No pipeline exists in current db
2025-06-17 21:41:28,632 [main] INFO recon.ReconServer: Initializing support of Recon Features...
2025-06-17 21:41:28,639 [main] INFO recon.ReconServer: Starting Recon server
2025-06-17 21:41:28,799 [main] INFO impl.MetricsConfig: Loaded properties from hadoop-metrics2.properties
2025-06-17 21:41:28,827 [main] INFO impl.MetricsSystemImpl: Scheduled Metric snapshot period at 10 second(s).
2025-06-17 21:41:28,828 [main] INFO impl.MetricsSystemImpl: Recon metrics system started
2025-06-17 21:41:29,212 [main] INFO http.HttpServer2: Jetty bound to port 9888
2025-06-17 21:41:29,214 [main] INFO server.Server: jetty-9.4.57.v20241219; built: 2025-01-08T21:24:30.412Z; git: df524e6b29271c2e09ba9aea83c18dc9db464a31; jvm 21.0.2+13-58
2025-06-17 21:41:29,290 [main] INFO server.session: DefaultSessionIdManager workerName=node0
2025-06-17 21:41:29,290 [main] INFO server.session: No SessionScavenger set, using defaults
2025-06-17 21:41:29,292 [main] INFO server.session: node0 Scavenging every 600000ms
2025-06-17 21:41:29,320 [main] INFO server.KerberosAuthenticationHandler: Using keytab /etc/security/keytabs/recon.keytab, for principal HTTP/recon@EXAMPLE.COM
2025-06-17 21:41:29,325 [main] INFO handler.ContextHandler: Started o.e.j.s.ServletContextHandler@1fd4850b{logs,/logs,file:///var/log/hadoop/,AVAILABLE}
2025-06-17 21:41:29,327 [main] INFO handler.ContextHandler: Started o.e.j.s.ServletContextHandler@7a085d02{static,/static,jar:file:/opt/hadoop/share/ozone/lib/ozone-recon-2.1.0-SNAPSHOT.jar!/webapps/static,AVAILABLE}
2025-06-17 21:41:29,616 [main] INFO server.KerberosAuthenticationHandler: Using keytab /etc/security/keytabs/recon.keytab, for principal HTTP/recon@EXAMPLE.COM
2025-06-17 21:41:29,629 [main] INFO server.KerberosAuthenticationHandler: Using keytab /etc/security/keytabs/recon.keytab, for principal HTTP/recon@EXAMPLE.COM
2025-06-17 21:41:31,962 [main] INFO handler.ContextHandler: Started o.e.j.w.WebAppContext@6a10f47a{recon,/,file:///tmp/ozone_http/jetty-0_0_0_0-9888-ozone-recon-2_1_0-SNAPSHOT_jar-_-any-15979289123936203548/webapp/,AVAILABLE}{jar:file:/opt/hadoop/share/ozone/lib/ozone-recon-2.1.0-SNAPSHOT.jar!/webapps/recon}
2025-06-17 21:41:31,983 [main] INFO server.AbstractConnector: Started ServerConnector@71b6edfb{HTTP/1.1, (http/1.1)}{0.0.0.0:9888}
2025-06-17 21:41:31,983 [main] INFO server.Server: Started @49047ms
2025-06-17 21:41:31,989 [main] INFO impl.MetricsSinkAdapter: Sink prometheus started
2025-06-17 21:41:31,989 [main] INFO impl.MetricsSystemImpl: Registered sink prometheus
2025-06-17 21:41:31,990 [main] INFO http.BaseHttpServer: HTTP server of recon listening at http://recon:9888
2025-06-17 21:41:31,990 [main] INFO impl.OzoneManagerServiceProviderImpl: Starting Ozone Manager Service Provider.
2025-06-17 21:41:31,991 [main] INFO recovery.ReconOmMetadataManagerImpl: Starting ReconOMMetadataManagerImpl
2025-06-17 21:41:31,991 [main] WARN recon.ReconUtils: ozone.recon.om.db.dir is not configured. We recommend adding this setting. Falling back to ozone.metadata.dirs instead.
2025-06-17 21:41:31,992 [main] INFO tasks.ReconTaskControllerImpl: Starting Recon Task Controller.
2025-06-17 21:41:32,060 [main] INFO scm.ReconStorageContainerManagerFacade: Recon ScmDatanodeProtocol RPC server is listening at /0.0.0.0:9891
2025-06-17 21:41:32,878 [main] INFO scm.ReconStorageContainerManagerFacade: Obtained 0 pipelines from SCM.
2025-06-17 21:41:32,881 [main] INFO scm.ReconPipelineManager: Recon has 0 pipelines in house.
2025-06-17 21:41:32,884 [main] INFO scm.ReconStorageContainerManagerFacade: SCM DB initialized
2025-06-17 21:41:32,888 [main] INFO server.SCMDatanodeProtocolServer: ScmDatanodeProtocol RPC server for DataNodes is listening at /0.0.0.0:9891
2025-06-17 21:41:32,944 [IPC Server Responder] INFO ipc.Server: IPC Server Responder: starting
2025-06-17 21:41:32,950 [IPC Server listener on 9891] INFO ipc.Server: IPC Server listener on 9891: starting
2025-06-17 21:41:52,061 [Recon-SyncOM-0] INFO impl.OzoneManagerServiceProviderImpl: Last known sequence number before sync: 0
2025-06-17 21:41:52,061 [Recon-SyncOM-0] INFO impl.OzoneManagerServiceProviderImpl: Seq number of Recon's OM DB : 0
2025-06-17 21:41:52,061 [Recon-SyncOM-0] INFO impl.OzoneManagerServiceProviderImpl: Obtaining full snapshot from Ozone Manager
2025-06-17 21:41:52,377 [Recon-SyncOM-0] INFO updater.ReconTaskStatusUpdater: Registered Task: OmSnapshotRequest
2025-06-17 21:41:52,383 [Recon-SyncOM-0] INFO impl.OzoneManagerServiceProviderImpl: Permissions for Recon DB directory '/data/metadata/recon' meet the minimum required permissions '750'
2025-06-17 21:41:56,923 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 3 failover attempts. Trying to failover immediately. Current retry count: 3.
2025-06-17 21:41:56,946 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 4 failover attempts. Trying to failover immediately. Current retry count: 4.
2025-06-17 21:41:56,947 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 5 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 5.
2025-06-17 21:41:58,948 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 6 failover attempts. Trying to failover immediately. Current retry count: 6.
2025-06-17 21:41:58,949 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 7 failover attempts. Trying to failover immediately. Current retry count: 7.
2025-06-17 21:41:58,957 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 8 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 8.
2025-06-17 21:42:00,959 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 9 failover attempts. Trying to failover immediately. Current retry count: 9.
2025-06-17 21:42:00,962 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 10 failover attempts. Trying to failover immediately. Current retry count: 10.
2025-06-17 21:42:00,967 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 11 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 11.
2025-06-17 21:42:02,968 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 12 failover attempts. Trying to failover immediately. Current retry count: 12.
2025-06-17 21:42:02,977 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 13 failover attempts. Trying to failover immediately. Current retry count: 13.
2025-06-17 21:42:02,988 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 14 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 14.
2025-06-17 21:42:04,992 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 15 failover attempts. Trying to failover immediately. Current retry count: 15.
2025-06-17 21:42:04,996 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 16 failover attempts. Trying to failover immediately. Current retry count: 16.
2025-06-17 21:42:05,001 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 17 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 17.
2025-06-17 21:42:07,008 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 18 failover attempts. Trying to failover immediately. Current retry count: 18.
2025-06-17 21:42:07,015 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 19 failover attempts. Trying to failover immediately. Current retry count: 19.
2025-06-17 21:42:07,020 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 20 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 20.
2025-06-17 21:42:09,024 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 21 failover attempts. Trying to failover immediately. Current retry count: 21.
2025-06-17 21:42:09,025 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 22 failover attempts. Trying to failover immediately. Current retry count: 22.
2025-06-17 21:42:09,030 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 23 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 23.
2025-06-17 21:42:11,037 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 24 failover attempts. Trying to failover immediately. Current retry count: 24.
2025-06-17 21:42:11,049 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 25 failover attempts. Trying to failover immediately. Current retry count: 25.
2025-06-17 21:42:11,060 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 26 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 26.
2025-06-17 21:42:13,068 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 27 failover attempts. Trying to failover immediately. Current retry count: 27.
2025-06-17 21:42:13,088 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 28 failover attempts. Trying to failover immediately. Current retry count: 28.
2025-06-17 21:42:13,100 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 29 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 29.
2025-06-17 21:42:15,103 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 30 failover attempts. Trying to failover immediately. Current retry count: 30.
2025-06-17 21:42:15,109 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 31 failover attempts. Trying to failover immediately. Current retry count: 31.
2025-06-17 21:42:15,122 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 32 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 32.
2025-06-17 21:42:17,128 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 33 failover attempts. Trying to failover immediately. Current retry count: 33.
2025-06-17 21:42:17,133 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 34 failover attempts. Trying to failover immediately. Current retry count: 34.
2025-06-17 21:42:17,134 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 35 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 35.
2025-06-17 21:42:19,137 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 36 failover attempts. Trying to failover immediately. Current retry count: 36.
2025-06-17 21:42:19,143 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 37 failover attempts. Trying to failover immediately. Current retry count: 37.
2025-06-17 21:42:19,147 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 38 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 38.
2025-06-17 21:42:21,150 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 39 failover attempts. Trying to failover immediately. Current retry count: 39.
2025-06-17 21:42:21,155 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 40 failover attempts. Trying to failover immediately. Current retry count: 40.
2025-06-17 21:42:21,157 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 41 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 41.
2025-06-17 21:42:23,163 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 42 failover attempts. Trying to failover immediately. Current retry count: 42.
2025-06-17 21:42:23,165 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 43 failover attempts. Trying to failover immediately. Current retry count: 43.
2025-06-17 21:42:23,167 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 44 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 44.
2025-06-17 21:42:25,169 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 45 failover attempts. Trying to failover immediately. Current retry count: 45.
2025-06-17 21:42:25,173 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 46 failover attempts. Trying to failover immediately. Current retry count: 46.
2025-06-17 21:42:25,179 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 47 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 47.
2025-06-17 21:42:27,181 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 48 failover attempts. Trying to failover immediately. Current retry count: 48.
2025-06-17 21:42:27,189 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 49 failover attempts. Trying to failover immediately. Current retry count: 49.
2025-06-17 21:42:27,191 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 50 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 50.
2025-06-17 21:42:29,195 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 51 failover attempts. Trying to failover immediately. Current retry count: 51.
2025-06-17 21:42:29,204 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 52 failover attempts. Trying to failover immediately. Current retry count: 52.
2025-06-17 21:42:29,207 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 53 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 53.
2025-06-17 21:42:30,851 [Socket Reader #2 for port 9891] INFO ipc.Server: Auth successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) from 172.25.0.104:56140
2025-06-17 21:42:30,865 [Socket Reader #2 for port 9891] INFO authorize.ServiceAuthorizationManager: Authorization successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) for protocol=interface org.apache.hadoop.ozone.protocol.ReconDatanodeProtocol
2025-06-17 21:42:31,231 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 54 failover attempts. Trying to failover immediately. Current retry count: 54.
2025-06-17 21:42:31,235 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 55 failover attempts. Trying to failover immediately. Current retry count: 55.
2025-06-17 21:42:31,251 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 56 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 56.
2025-06-17 21:42:32,643 [IPC Server handler 2 on default port 9891] INFO net.NetworkTopologyImpl: Added a new node: /default-rack/0e566885-7bcf-4ab2-9619-b33fd79adf32
2025-06-17 21:42:32,663 [IPC Server handler 2 on default port 9891] INFO node.SCMNodeManager: Registered datanode: 0e566885-7bcf-4ab2-9619-b33fd79adf32{ip: 172.25.0.104, host: ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net, ports: [HTTP=9882, CLIENT_RPC=19864, REPLICATION=9886, RATIS=9858, RATIS_ADMIN=9857, RATIS_SERVER=9856, RATIS_DATASTREAM=9855, STANDALONE=9859], networkLocation: /default-rack, certSerialId: 6, persistedOpState: null, persistedOpStateExpiryEpochSec: 0}
2025-06-17 21:42:32,665 [IPC Server handler 2 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:32,960 [Socket Reader #3 for port 9891] INFO ipc.Server: Auth successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) from 172.25.0.103:53586
2025-06-17 21:42:32,969 [Socket Reader #3 for port 9891] INFO authorize.ServiceAuthorizationManager: Authorization successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) for protocol=interface org.apache.hadoop.ozone.protocol.ReconDatanodeProtocol
2025-06-17 21:42:33,265 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 57 failover attempts. Trying to failover immediately. Current retry count: 57.
2025-06-17 21:42:33,266 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 58 failover attempts. Trying to failover immediately. Current retry count: 58.
2025-06-17 21:42:33,274 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 59 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 59.
2025-06-17 21:42:33,510 [Recon-EventQueue-NewNodeForReconNewNodeHandler] INFO scm.ReconNodeManager: Added a new node to db: 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:33,644 [Socket Reader #4 for port 9891] INFO ipc.Server: Auth successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) from 172.25.0.102:60614
2025-06-17 21:42:33,657 [Socket Reader #4 for port 9891] INFO authorize.ServiceAuthorizationManager: Authorization successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) for protocol=interface org.apache.hadoop.ozone.protocol.ReconDatanodeProtocol
2025-06-17 21:42:34,524 [IPC Server handler 6 on default port 9891] INFO scm.ReconNodeManager: Sending ReregisterCommand() for 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:34,775 [IPC Server handler 8 on default port 9891] INFO net.NetworkTopologyImpl: Added a new node: /default-rack/0c683d87-9b58-4caa-ba47-c3e43235e896
2025-06-17 21:42:34,775 [IPC Server handler 8 on default port 9891] INFO node.SCMNodeManager: Registered datanode: 0c683d87-9b58-4caa-ba47-c3e43235e896{ip: 172.25.0.103, host: ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net, ports: [HTTP=9882, CLIENT_RPC=19864, REPLICATION=9886, RATIS=9858, RATIS_ADMIN=9857, RATIS_SERVER=9856, RATIS_DATASTREAM=9855, STANDALONE=9859], networkLocation: /default-rack, certSerialId: 7, persistedOpState: null, persistedOpStateExpiryEpochSec: 0}
2025-06-17 21:42:34,776 [IPC Server handler 8 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:34,779 [Recon-EventQueue-NewNodeForReconNewNodeHandler] INFO scm.ReconNodeManager: Added a new node to db: 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:35,284 [IPC Server handler 2 on default port 9891] INFO net.NetworkTopologyImpl: Added a new node: /default-rack/38b3ef17-21f0-435d-b8f7-5fbe2b0febf7
2025-06-17 21:42:35,284 [IPC Server handler 2 on default port 9891] INFO node.SCMNodeManager: Registered datanode: 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7{ip: 172.25.0.102, host: ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net, ports: [HTTP=9882, CLIENT_RPC=19864, REPLICATION=9886, RATIS=9858, RATIS_ADMIN=9857, RATIS_SERVER=9856, RATIS_DATASTREAM=9855, STANDALONE=9859], networkLocation: /default-rack, certSerialId: 8, persistedOpState: null, persistedOpStateExpiryEpochSec: 0}
2025-06-17 21:42:35,288 [IPC Server handler 2 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:35,289 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 60 failover attempts. Trying to failover immediately. Current retry count: 60.
2025-06-17 21:42:35,290 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 61 failover attempts. Trying to failover immediately. Current retry count: 61.
2025-06-17 21:42:35,291 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 62 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 62.
2025-06-17 21:42:35,295 [Recon-EventQueue-NewNodeForReconNewNodeHandler] INFO scm.ReconNodeManager: Added a new node to db: 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:36,588 [IPC Server handler 8 on default port 9891] INFO scm.ReconNodeManager: Sending ReregisterCommand() for 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:37,263 [IPC Server handler 2 on default port 9891] INFO scm.ReconNodeManager: Sending ReregisterCommand() for 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:37,295 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 63 failover attempts. Trying to failover immediately. Current retry count: 63.
2025-06-17 21:42:37,301 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 64 failover attempts. Trying to failover immediately. Current retry count: 64.
2025-06-17 21:42:37,304 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 65 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 65.
2025-06-17 21:42:38,487 [IPC Server handler 8 on default port 9891] INFO scm.ReconNodeManager: Updated nodes db table for 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:38,495 [IPC Server handler 8 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:38,500 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Unknown pipeline Pipeline-154ab724-7a15-47a3-94aa-5a8fb917b7b8. Trying to get from SCM.
2025-06-17 21:42:38,608 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline Pipeline{ Id: 154ab724-7a15-47a3-94aa-5a8fb917b7b8, Nodes: [ {0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103), ReplicaIndex: 0},], ReplicationConfig: RATIS/ONE, State:OPEN, leaderId:0c683d87-9b58-4caa-ba47-c3e43235e896, CreationTimestamp2025-06-17T21:42:34.775Z[UTC]} verified from SCM and added to Recon pipeline metadata.
2025-06-17 21:42:38,614 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Unknown pipeline Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80. Trying to get from SCM.
2025-06-17 21:42:38,643 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline Pipeline{ Id: ee9e54cb-5f3c-4eca-9e19-baedd735eb80, Nodes: [ {38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102), ReplicaIndex: 0}, {0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103), ReplicaIndex: 0}, {0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104), ReplicaIndex: 0},], ReplicationConfig: RATIS/THREE, State:ALLOCATED, leaderId:, CreationTimestamp2025-06-17T21:42:35.403Z[UTC]} verified from SCM and added to Recon pipeline metadata.
2025-06-17 21:42:38,648 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:39,236 [IPC Server handler 2 on default port 9891] INFO scm.ReconNodeManager: Updated nodes db table for 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:39,237 [IPC Server handler 2 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:39,239 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:39,240 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Unknown pipeline Pipeline-df9e0501-d723-4db8-b4b7-87997bd5fb06. Trying to get from SCM.
2025-06-17 21:42:39,265 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline Pipeline{ Id: df9e0501-d723-4db8-b4b7-87997bd5fb06, Nodes: [ {38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102), ReplicaIndex: 0},], ReplicationConfig: RATIS/ONE, State:OPEN, leaderId:38b3ef17-21f0-435d-b8f7-5fbe2b0febf7, CreationTimestamp2025-06-17T21:42:35.286Z[UTC]} verified from SCM and added to Recon pipeline metadata.
2025-06-17 21:42:39,307 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 66 failover attempts. Trying to failover immediately. Current retry count: 66.
2025-06-17 21:42:39,319 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 67 failover attempts. Trying to failover immediately. Current retry count: 67.
2025-06-17 21:42:39,330 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 68 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 68.
2025-06-17 21:42:41,331 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 69 failover attempts. Trying to failover immediately. Current retry count: 69.
2025-06-17 21:42:41,334 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 70 failover attempts. Trying to failover immediately. Current retry count: 70.
2025-06-17 21:42:41,338 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 71 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 71.
2025-06-17 21:42:41,454 [IPC Server handler 8 on default port 9891] INFO scm.ReconNodeManager: Updated nodes db table for 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:41,459 [IPC Server handler 8 on default port 9891] INFO recon.ReconContext: Update healthStatus of Recon from true to true.
2025-06-17 21:42:41,465 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Unknown pipeline Pipeline-9d6153d1-679f-4c5d-9b17-735e3785ab12. Trying to get from SCM.
2025-06-17 21:42:41,497 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline Pipeline{ Id: 9d6153d1-679f-4c5d-9b17-735e3785ab12, Nodes: [ {0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104), ReplicaIndex: 0},], ReplicationConfig: RATIS/ONE, State:OPEN, leaderId:0e566885-7bcf-4ab2-9619-b33fd79adf32, CreationTimestamp2025-06-17T21:42:32.863Z[UTC]} verified from SCM and added to Recon pipeline metadata.
2025-06-17 21:42:42,413 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:42,639 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:42,640 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Unknown pipeline Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c. Trying to get from SCM.
2025-06-17 21:42:42,659 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline Pipeline{ Id: 54dd23fc-2662-4882-9c31-35767bb9160c, Nodes: [ {0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103), ReplicaIndex: 0}, {38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102), ReplicaIndex: 0}, {0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104), ReplicaIndex: 0},], ReplicationConfig: RATIS/THREE, State:ALLOCATED, leaderId:, CreationTimestamp2025-06-17T21:42:35.509Z[UTC]} verified from SCM and added to Recon pipeline metadata.
2025-06-17 21:42:42,663 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:42,784 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:42,786 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:43,285 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:43,287 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:43,342 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 72 failover attempts. Trying to failover immediately. Current retry count: 72.
2025-06-17 21:42:43,345 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 73 failover attempts. Trying to failover immediately. Current retry count: 73.
2025-06-17 21:42:43,349 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 74 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 74.
2025-06-17 21:42:43,524 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:43,524 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 0c683d87-9b58-4caa-ba47-c3e43235e896(ozonesecure-ha-datanode2-1.ozonesecure-ha_ozone_net/172.25.0.103)
2025-06-17 21:42:44,265 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:44,265 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:45,351 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 75 failover attempts. Trying to failover immediately. Current retry count: 75.
2025-06-17 21:42:45,356 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 76 failover attempts. Trying to failover immediately. Current retry count: 76.
2025-06-17 21:42:45,358 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 77 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 77.
2025-06-17 21:42:47,360 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 78 failover attempts. Trying to failover immediately. Current retry count: 78.
2025-06-17 21:42:47,361 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 79 failover attempts. Trying to failover immediately. Current retry count: 79.
2025-06-17 21:42:47,362 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 80 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 80.
2025-06-17 21:42:48,141 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:48,145 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-54dd23fc-2662-4882-9c31-35767bb9160c reported by 38b3ef17-21f0-435d-b8f7-5fbe2b0febf7(ozonesecure-ha-datanode1-1.ozonesecure-ha_ozone_net/172.25.0.102)
2025-06-17 21:42:48,599 [Recon-EventQueue-PipelineReportForReconPipelineReportHandler] INFO scm.ReconPipelineReportHandler: Pipeline RATIS/THREE Pipeline-ee9e54cb-5f3c-4eca-9e19-baedd735eb80 reported by 0e566885-7bcf-4ab2-9619-b33fd79adf32(ozonesecure-ha-datanode3-1.ozonesecure-ha_ozone_net/172.25.0.104)
2025-06-17 21:42:49,363 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 81 failover attempts. Trying to failover immediately. Current retry count: 81.
2025-06-17 21:42:49,368 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 82 failover attempts. Trying to failover immediately. Current retry count: 82.
2025-06-17 21:42:49,372 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 83 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 83.
2025-06-17 21:42:51,375 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 84 failover attempts. Trying to failover immediately. Current retry count: 84.
2025-06-17 21:42:51,381 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 85 failover attempts. Trying to failover immediately. Current retry count: 85.
2025-06-17 21:42:51,386 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 86 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 86.
2025-06-17 21:42:53,390 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 87 failover attempts. Trying to failover immediately. Current retry count: 87.
2025-06-17 21:42:53,395 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 88 failover attempts. Trying to failover immediately. Current retry count: 88.
2025-06-17 21:42:53,399 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 89 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 89.
2025-06-17 21:42:55,405 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om1:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 90 failover attempts. Trying to failover immediately. Current retry count: 90.
2025-06-17 21:42:55,410 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om2:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 91 failover attempts. Trying to failover immediately. Current retry count: 91.
2025-06-17 21:42:55,413 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: java.net.ConnectException: Call From recon/172.25.0.115 to om3:9862 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 92 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 92.
2025-06-17 21:43:01,602 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om1 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 93 failover attempts. Trying to failover immediately. Current retry count: 93.
2025-06-17 21:43:01,903 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om2 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 94 failover attempts. Trying to failover immediately. Current retry count: 94.
2025-06-17 21:43:02,036 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om3 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 95 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 95.
2025-06-17 21:43:04,039 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om1 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 96 failover attempts. Trying to failover immediately. Current retry count: 96.
2025-06-17 21:43:04,043 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om2 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 97 failover attempts. Trying to failover immediately. Current retry count: 97.
2025-06-17 21:43:04,045 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om3 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 98 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 98.
2025-06-17 21:43:06,051 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om1 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om1,nodeAddress=om1:9862 after 99 failover attempts. Trying to failover immediately. Current retry count: 99.
2025-06-17 21:43:06,061 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMNotLeaderException): OM:om2 is not the leader. Could not determine the leader node.
	at org.apache.hadoop.ozone.om.ratis.OzoneManagerRatisServer.newOMNotLeaderException(OzoneManagerRatisServer.java:810)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:200)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om2,nodeAddress=om2:9862 after 100 failover attempts. Trying to failover immediately. Current retry count: 100.
2025-06-17 21:43:06,076 [Recon-SyncOM-0] INFO retry.RetryInvocationHandler: com.google.protobuf.ServiceException: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ozone.om.exceptions.OMLeaderNotReadyException): om3 is Leader but not ready to process request yet.
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderNotReadyException(OzoneManagerProtocolServerSideTranslatorPB.java:210)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.createLeaderErrorException(OzoneManagerProtocolServerSideTranslatorPB.java:202)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitReadRequestToOM(OzoneManagerProtocolServerSideTranslatorPB.java:193)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.internalProcessRequest(OzoneManagerProtocolServerSideTranslatorPB.java:158)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.processRequest(OzoneManagerProtocolServerSideTranslatorPB.java:124)
	at org.apache.hadoop.hdds.server.OzoneProtocolMessageDispatcher.processRequest(OzoneProtocolMessageDispatcher.java:87)
	at org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB.submitRequest(OzoneManagerProtocolServerSideTranslatorPB.java:115)
	at org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos$OzoneManagerService$2.callBlockingMethod(OzoneManagerProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.processCall(ProtobufRpcEngine.java:484)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:595)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine2.java:573)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1227)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1246)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1169)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:3198)
, while invoking $Proxy56.submitRequest over nodeId=om3,nodeAddress=om3:9862 after 101 failover attempts. Trying to failover after sleeping for 2000ms. Current retry count: 101.
2025-06-17 21:43:08,558 [Recon-SyncOM-0] ERROR impl.OzoneManagerServiceProviderImpl: Unable to update Recon's metadata with new OM DB. 
java.lang.reflect.UndeclaredThrowableException
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1969)
	at org.apache.hadoop.security.SecurityUtil.doAsUser(SecurityUtil.java:555)
	at org.apache.hadoop.security.SecurityUtil.doAsLoginUser(SecurityUtil.java:535)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.getOzoneManagerDBSnapshot(OzoneManagerServiceProviderImpl.java:443)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.updateReconOmDBWithNewSnapshot(OzoneManagerServiceProviderImpl.java:490)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.syncDataFromOM(OzoneManagerServiceProviderImpl.java:704)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.lambda$startSyncDataFromOM$2(OzoneManagerServiceProviderImpl.java:325)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.apache.hadoop.security.authentication.client.AuthenticationException: Error while authenticating with endpoint: http://om3:9874/dbCheckpoint
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator.wrapExceptionWithMessage(KerberosAuthenticator.java:237)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator.authenticate(KerberosAuthenticator.java:220)
	at org.apache.hadoop.security.authentication.client.AuthenticatedURL.openConnection(AuthenticatedURL.java:353)
	at org.apache.hadoop.hdfs.web.URLConnectionFactory.openConnection(URLConnectionFactory.java:186)
	at org.apache.hadoop.ozone.recon.ReconUtils.makeHttpCall(ReconUtils.java:421)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.lambda$getOzoneManagerDBSnapshot$4(OzoneManagerServiceProviderImpl.java:444)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)
	... 12 more
Caused by: org.apache.hadoop.security.authentication.client.AuthenticationException: GSSException: No valid credentials provided (Mechanism level: Server not found in Kerberos database (7) - LOOKING_UP_SERVER)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator.doSpnegoSequence(KerberosAuthenticator.java:368)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator.authenticate(KerberosAuthenticator.java:205)
	... 19 more
Caused by: GSSException: No valid credentials provided (Mechanism level: Server not found in Kerberos database (7) - LOOKING_UP_SERVER)
	at java.security.jgss/sun.security.jgss.krb5.Krb5Context.initSecContext(Krb5Context.java:762)
	at java.security.jgss/sun.security.jgss.GSSContextImpl.initSecContext(GSSContextImpl.java:266)
	at java.security.jgss/sun.security.jgss.GSSContextImpl.initSecContext(GSSContextImpl.java:196)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator$1.run(KerberosAuthenticator.java:344)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator$1.run(KerberosAuthenticator.java:318)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/javax.security.auth.Subject.doAs(Subject.java:525)
	at org.apache.hadoop.security.authentication.client.KerberosAuthenticator.doSpnegoSequence(KerberosAuthenticator.java:318)
	... 20 more
Caused by: KrbException: Server not found in Kerberos database (7) - LOOKING_UP_SERVER
	at java.security.jgss/sun.security.krb5.KrbTgsRep.<init>(KrbTgsRep.java:72)
	at java.security.jgss/sun.security.krb5.KrbTgsReq.sendAndGetCreds(KrbTgsReq.java:215)
	at java.security.jgss/sun.security.krb5.internal.CredentialsUtil.serviceCredsSingle(CredentialsUtil.java:473)
	at java.security.jgss/sun.security.krb5.internal.CredentialsUtil.serviceCreds(CredentialsUtil.java:338)
	at java.security.jgss/sun.security.krb5.internal.CredentialsUtil.acquireServiceCreds(CredentialsUtil.java:163)
	at java.security.jgss/sun.security.krb5.Credentials.acquireServiceCreds(Credentials.java:491)
	at java.security.jgss/sun.security.jgss.krb5.Krb5Context.initSecContext(Krb5Context.java:690)
	... 27 more
Caused by: KrbException: Identifier doesn't match expected value (906)
	at java.security.jgss/sun.security.krb5.internal.KDCRep.init(KDCRep.java:140)
	at java.security.jgss/sun.security.krb5.internal.TGSRep.init(TGSRep.java:65)
	at java.security.jgss/sun.security.krb5.internal.TGSRep.<init>(TGSRep.java:60)
	at java.security.jgss/sun.security.krb5.KrbTgsRep.<init>(KrbTgsRep.java:54)
	... 33 more
2025-06-17 21:43:08,561 [Recon-SyncOM-0] INFO recon.ReconContext: Update healthStatus of Recon from true to false.
2025-06-17 21:43:08,562 [Recon-SyncOM-0] ERROR impl.OzoneManagerServiceProviderImpl: Unexpected exception while syncing data from OM.
java.lang.IllegalArgumentException: Unknown table fileTable
	at org.apache.hadoop.ozone.om.OmMetadataManagerImpl.getTable(OmMetadataManagerImpl.java:1669)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.printTableCount(OzoneManagerServiceProviderImpl.java:766)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.printOMDBMetaInfo(OzoneManagerServiceProviderImpl.java:761)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.syncDataFromOM(OzoneManagerServiceProviderImpl.java:749)
	at org.apache.hadoop.ozone.recon.spi.impl.OzoneManagerServiceProviderImpl.lambda$startSyncDataFromOM$2(OzoneManagerServiceProviderImpl.java:325)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-06-17 21:43:13,532 [Socket Reader #5 for port 9891] INFO ipc.Server: Auth successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) from 172.25.0.103:45688
2025-06-17 21:43:13,537 [Socket Reader #5 for port 9891] INFO authorize.ServiceAuthorizationManager: Authorization successful for dn/dn@EXAMPLE.COM (auth:KERBEROS) for protocol=interface org.apache.hadoop.ozone.protocol.ReconDatanodeProtocol
